{
    "title": "291 genes improved expression by 2000 IU of vitamin D \u2013 RCT",
    "slug": "291-genes-improved-expression-by-2000-iu-of-vitamin-d-rct",
    "aliases": [
        "/291+genes+improved+expression+by+2000+IU+of+vitamin+D+\u2013+RCT+March+2013",
        "/3926"
    ],
    "tiki_page_id": 3926,
    "date": "2013-03-21",
    "categories": [
        "Genetics",
        "Top news"
    ],
    "tags": [
        "Genetics",
        "Top news",
        "autoimmune",
        "blood levels",
        "cancer",
        "dosage",
        "high dose",
        "intervention",
        "therapeutic intervention",
        "vitamin d",
        "vitamin d blood test"
    ]
}


#### Influence of Vitamin D Status and Vitamin D3 Supplementation on Genome Wide Expression of White Blood Cells: A Randomized Double-Blind Clinical Trial

PLoS ONE 8(3): e58725. doi:10.1371/journal.pone.0058725

Arash Hossein-nezhad,     Avrum Spira,     Michael F. Holick

Background: Although there have been numerous observations of vitamin D deficiency and its links to chronic diseases, no studies have reported on how vitamin D status and vitamin D3 supplementation affects broad gene expression in humans. The objective of this study was to determine the effect of vitamin D status and subsequent vitamin D supplementation on broad gene expression in healthy adults. (Trial registration: ClinicalTrials.gov NCT01696409).

Methods and Findings: A randomized, double-blind, single center pilot trial was conducted for comparing vitamin D supplementation with either 400 IUs (n = 3) or  **2000**  IUs (n = 5) vitamin D3 daily for 2 months on broad gene expression in the white blood cells collected from 8 healthy adults in the winter. Microarrays of the 16 buffy coats from eight subjects passed the quality control filters and normalized with the RMA method. 

Vitamin D3 supplementation that improved serum 25-hydroxyvitamin D concentrations was associated with at least a  **1.5 fold alteration in the expression of 291 genes** . 

There was a significant difference in the expression of 66 genes between subjects at baseline with vitamin D deficiency (25(OH)D<20 ng/ml) and subjects with a 25(OH)D>20 ng/ml. 

After vitamin D3 supplementation gene expression of these 66 genes was similar for both groups. 

Seventeen vitamin D-regulated genes with new candidate vitamin D response elements including 

* TRIM27, 

* CD83, 

* COPB2, 

* YRNA and 

* CETN3 

which have been shown to be important for transcriptional regulation, immune function, response to stress and DNA repair were identified.

Conclusion/Significance: Our data suggest that any improvement in vitamin D status will significantly affect expression of genes that have a wide variety of biologic functions of more than 160 pathways linked to cancer, autoimmune disorders and cardiovascular disease with have been associated with vitamin D deficiency. 

 **This study reveals for the first time molecular finger prints that help explain the nonskeletal health benefits of vitamin D.** 

-=A few items from the study=-

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/3a.jpg" alt="image">

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/3b.jpg" alt="image">

 **Figure 3. Heatmaps of vitamin D responsive genes whose expression levels change after 2 months vitamin D3 supplementation.**  

Before supplementation  **<span style="color:#090;">(light green) </span>**  four subjects were vitamin D deficient with 25(OH)D of 16.264.2 ng/ml  **<span style="color:#900;">(dark purple)</span>**  and the other four subjects were insufficient or sufficient with a 25(OH)D of 27.568.4 ng/ml  **<span style="color:#90F;">(light purple)</span>** . After supplementation (dark green) serum levels of 25(OH)D in vitamin D insufficient/sufficient subjects increased to 35.268.2 ng/ml (light purple) and in the vitamin deficient subjects increased to 25.1 6 4.7 ng/ml  **<span style="color:#900;">(dark purple)</span>**  Two groups of gene-expression changes are seen based on stimulation (brown) or inhibition (blue) of gene expression post vitamin D3 supplementation. (Colors ranged from blue to brown; High expression = brown, average expression = white, low expression = blue). Clustering of the 291 genes affected by vitamin D3 supplementation was based on stimulation (brown) or  **<span style="color:#00F;">inhibition (blue)</span>**  of gene expression. The list of the 291 genes is shown in Table S1. doi:10.1371/journal.pone.0058725.g003 that was stimulated 1.6 fold by vitamin D3 supplementation has five VDREs (Table 2) that one of them is shown in Figure 6D located at position -1027, the TATA box located at -276 and location of other transcription factor sites near this VDRE was determined (6D).

---

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/5a.jpg" alt="image">

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/5b.jpg" alt="image">

 **Figure 5. Heatmaps of vitamin D responsive genes affected by vitamin D status.** 

Before supplementation (light green) four subjects were vitamin D deficient with 25(OH)D of 16.264.2 ng/ml  **<span style="color:#900;">(dark purple)</span>**  and the other four subjects were insufficient or sufficient with a 25(OH)D of 27.568.4 ng/ml  **<span style="color:#90F;">(light purple)</span>** . After supplementation (dark green) serum levels of 25(OH)D in vitamin D insufficient/sufficient subjects increased to 35.268.2 ng/ml (light purple) and in the vitamin deficient subjects increased to 25(OH)D of 25.164.7 ng/ml  **<span style="color:#90F;">(light purple)</span>** . Two groups of gene-expression changes are seen based on stimulation (brown) or i **<span style="color:#00F;">inhibition (blue)</span>**  of gene expression post vitamin D3 supplementation. (Colors ranged from blue to brown; High expression = brown, average expression = white, low expression = blue).Expression of 66 genes before supplementation was significantly different in the vitamin D deficient group (dark purple) compared to the vitamin D insufficient/sufficient group (light purple). Clustering of the 66 genes affected by vitamin D status and vitamin D3 supplementation was based on stimulation (brown) or inhibition (blue) of gene expression.

---

<img src="/attachments/d3.mock.jpg" alt="image"> 

 **Figure 7. Biological functions for genes whose expression levels were altered after 2 months of vitamin D3 supplementation.** 

After receiving vitamin D3 supplementation we identified 291 genes whose expression was significantly decreased or increased.  

Some of these genes influence several pathways that are involved in response to stres s and DNA repair, DNA replication, immune regulation, epigenetic modification,transcriptional regulation and other biological functions.In addition vitamin D3 supplementation influenced the expression of Y RNA and CETN3 that are involved in DNA repair in response to UVR exposure.

---

 **Table 3. List of biological functions of the 291 genes whose expression was influenced by vitamin D3 supplementation.** 

| | |
| --- | --- |
| Biological functions | Gene symbol |
| Apoptosis and immune response | OSM, AXUD1, CD83, PHLPP, TNFAIP3,NFKBIA,ZNF287, PTRH2, XIAP, TNFAIP8L2, ZDHHC16, TIA1,NUDCD1, KLF11,RASA1,EGR1,  |
| Mineralization and bone development | JUND, SBDS, ZNRD1,MINPP1 |
| Transcriptional regulation | ORC2L,KLF10,TRIM27,EGR1,NFIL3,JUN,NR4A2,ZNF225,ZNF607,ZNF780B,ZNF616RASA1,ZNF397, ZNF284,ZFP62,HOMEZ,ZNF701, GTF2E1,ZNF232, ZNF473,TAF1A, ZNF587,MIZF,ZNF223, ZNF175,MED7, ZNF320,ZNF17,ZNF45,ZFP3,ZNF283, EGR1,MED17,ZNF235,NF780A,ZNF322A, KLF11, SUV420H1,ZNF852, HCFC2,NAPC3,TRIP11,JRKL,ZNF234,ZNF260,JUNB, KLF10, TRIM27 |
| Metabolic processes | TGDS, NAPEPLD, KIAA0859, ARSK, TMEM68INSIG2, GALK2, FPGT, HMGCL, HSD17B7P2, HSD17B7 |
| Response to stress and DNA repair | FANCF, MSH5, PXDNL,ATF4,STIP1,HSPA4,HSPH1,POLA2, SOS1 |
| RNA processing | CCDC76, MTO1, C1orf25, PUS3, RBM5, CSTF3, ZFP36, RNASEL, NUP107, ZCCHC8,POP1,INTS7, GEMIN6 |
| Ion and protein transporter | SLC39A7, SLC30A6, SLC30A5, COPB2, NUP43, GOPC, SLC35B3, BET1, USO1, PIGM, TRAPPC6B |
| Biological process | TGDS, NAPEPLD, KIAA0859, ARSK, TMEM68, INSIG2, GALK2, FPGT, HMGCL, HSD17B7P2, HSD17B7 |
| Epigenetic modification | HIST1H1E, H1FX, ALKBH1,UTP3, N6AMT1, METTL4 |
| Cell cycle and DNA replication | FGF5, IRS2, MIS12, C10orf2, ORC2L, HELB, CUZD1, KIAA1009, POLA2, CETN3,CEP110, POLA2, PTP4A1 |
| Signal transduction | GRASP, GNRH1, TAS2R4, TAS2R3, CXCR7, RIC8B, SOS1, BBS10 |
| Protein modification | PIM3, PTP4A1, RNF139, PDP2, GGCX, PPID, TTC9C, SIK1, STK38 |
| Development and cell differentiation | PDE4DIP, TUBD1, KEAP1, BBS7, UTP3, C11orf73, MKKS, BPNT1,NOC3L |