---
title: "CYP2R1 gene reduces response to Vitamin D - many studies"
slug: "cyp2r1-gene-reduces-response-to-vitamin-d-many-studies"
aliases:
  - "/CYP2R1+gene+reduces+response+to+Vitamin+D+-+many+studies"
  - "/tiki-index.php?page_id=14254"
date: 2023-02-10
categories:
  - "Genetics"
  - "Stroke"
---


 **People with a poor CYP2R1 genes are poor Vitamin D responders%%% &nbsp;&nbsp; Possible solution: take larger doses of Vitamin D**  

<!-- {maketoc} -->
 

---

#### VitaminDWiki - [Genetics](/categories/genetics) chart shows the vitamin D genes

<img src="/attachments/d3.mock.jpg" alt="image" width="300">

---

#### Genes and other reasons for poor blood response to Vitamin D supplementation

* <a href="/posts/poor-or-no-response-to-vitamin-d-was-associated-with-poor-genes-cystic-fibrosis-4-genes" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022">Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022</a>

* <a href="/posts/reasons-for-low-response-to-vitamin-d" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Reasons for low response to vitamin D">Reasons for low response to vitamin D</a>

* <a href="/posts/autistics-have-half-of-the-response-to-vitamin-d-rct" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Autistics have half of the response to Vitamin D – RCT Oct 2018">Autistics have half of the response to Vitamin D – RCT Oct 2018</a>

* <a href="/posts/poor-cyp2r1-gene-reduces-blood-response-to-vitamin-d-supplementation" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019">Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019</a>

* <a href="/posts/poor-vitamin-d-response-4x-more-likely-if-poor-vitamin-d-binding-proteins" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019">Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019</a>

* <a href="/posts/vitamin-d-nutrigenomics-high-medium-and-low-responders" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019">Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019</a>

* <a href="/posts/vitamin-d-non-responders-may-have-one-or-more-poor-genes-gc-lipc-cyp24a1-and-pde3b" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022">Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022</a>

---

#### 24+ VitaminDWiki pages with CYP2R1 in title

This list is automatically updated

{LIST()}

{list max="50"}

<!-- {filter field="title" content="CYP2R1"} -->

<!-- {filter type="wiki page"} -->

{sort mode="modification_date_major_desc"}

{OUTPUT(template="table")}

   {tableparams allowtableexpansion="n" shownbitems="y" title=""}

    {column sort="title" label="Title" field="title" mode="raw"}{column}

    {column label="Modified" sort="modification_date_major" field="modification_date_major"}

{OUTPUT}

{FORMAT(name="title")}{display name=title format="objectlink"}{FORMAT}

{FORMAT(name="modification_date_major")}{display name=modification_date_major format="date"}{FORMAT}

{LIST}

---

#### CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023

 **Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke** 

Stroke . 2023 Feb 10. [doi: 10.1161/STROKEAHA.122.040883](https://doi.org/10.1161/STROKEAHA.122.040883)  **[<i class="fas fa-file-pdf" style="margin-right: 0.3em;"></i>PDF of Preprint](/attachments/d3.mock.pdf)** 

Yuhan Xie # 1, Julián N Acosta # 2, Yixuan Ye 3, Zachariah S Demarais 4, Carolyn J Conlon 4, Ming Chen 1, Hongyu Zhao 1 3, Guido J Falcone 2

<img src="/attachments/d3.mock.jpg" alt="image" width="800">

Background: Ischemic stroke (IS) is a highly heritable trait, and genome-wide association studies have identified several commonly occurring susceptibility risk loci for this condition. However, there are limited data on the contribution of rare genetic variation to IS.

Methods: We conducted an exome-wide study using whole-exome sequencing data from  **152 058 UK Biobank participants** , including 1777 IS cases. We performed single-variant analyses for rare variants and gene-based analyses for loss-of-function and deleterious missense rare variants. We validated these results through

* (1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, 

* (2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and

* (3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.

Results: In single-variant association analyses, none of the evaluated variants were associated with IS at genome-wide significance levels (P<5×10-8). In the gene-based analysis focused on loss-of-function and deleterious missense variants, rare genetic variation at CYP2R1 was significantly associated with IS risk (P=2.6×10-6), exceeding the Bonferroni-corrected threshold for 16 074 tests (P<3.1×10-6). Validations analyses indicated that  **CYP2R1**  was associated with IS risk in MEGASTROKE (gene-based test, P=0.003), with IS recurrence in the VISP trial (gene-based test, P=0.001) and with neuroimaging traits (white matter hyperintensity, mean diffusivity, and fractional anisotropy) in the UK Biobank neuroimaging study (all gene-based tests, P<0.05).

Conclusions: Because  **CYP2R1**  plays an important role in vitamin D metabolism and existing observational evidence suggests an association between vitamin D levels and cerebrovascular disease, our results support a role of this pathway in the occurrence of IS.

<!-- ~tc~ (alias(CYP2R1 gene reduces Vitamin D - many studies)) ~/tc~ -->


<pre style="background-color: #e0e0e0;">
<code class="language-text">
Markdown:
--------
 **People with a poor CYP2R1 genes are poor Vitamin D responders%%% &amp;nbsp;&amp;nbsp; Possible solution: take larger doses of Vitamin D**  

&lt;!-- {maketoc} --&gt;
 

---

#### VitaminDWiki - [Genetics](/categories/genetics) chart shows the vitamin D genes

&lt;img src=&quot;/attachments/d3.mock.jpg&quot; alt=&quot;image&quot; width=&quot;300&quot;&gt;

---

#### Genes and other reasons for poor blood response to Vitamin D supplementation

* &lt;a href=&quot;/posts/poor-or-no-response-to-vitamin-d-was-associated-with-poor-genes-cystic-fibrosis-4-genes&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022&quot;&gt;Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022&lt;/a&gt;

* &lt;a href=&quot;/posts/reasons-for-low-response-to-vitamin-d&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Reasons for low response to vitamin D&quot;&gt;Reasons for low response to vitamin D&lt;/a&gt;

* &lt;a href=&quot;/posts/autistics-have-half-of-the-response-to-vitamin-d-rct&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Autistics have half of the response to Vitamin D – RCT Oct 2018&quot;&gt;Autistics have half of the response to Vitamin D – RCT Oct 2018&lt;/a&gt;

* &lt;a href=&quot;/posts/poor-cyp2r1-gene-reduces-blood-response-to-vitamin-d-supplementation&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019&quot;&gt;Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019&lt;/a&gt;

* &lt;a href=&quot;/posts/poor-vitamin-d-response-4x-more-likely-if-poor-vitamin-d-binding-proteins&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019&quot;&gt;Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019&lt;/a&gt;

* &lt;a href=&quot;/posts/vitamin-d-nutrigenomics-high-medium-and-low-responders&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019&quot;&gt;Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019&lt;/a&gt;

* &lt;a href=&quot;/posts/vitamin-d-non-responders-may-have-one-or-more-poor-genes-gc-lipc-cyp24a1-and-pde3b&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022&quot;&gt;Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022&lt;/a&gt;

---

#### 24+ VitaminDWiki pages with CYP2R1 in title

This list is automatically updated

{LIST()}

{list max=&quot;50&quot;}

&lt;!-- {filter field=&quot;title&quot; content=&quot;CYP2R1&quot;} --&gt;

&lt;!-- {filter type=&quot;wiki page&quot;} --&gt;

{sort mode=&quot;modification_date_major_desc&quot;}

{OUTPUT(template=&quot;table&quot;)}

   {tableparams allowtableexpansion=&quot;n&quot; shownbitems=&quot;y&quot; title=&quot;&quot;}

    {column sort=&quot;title&quot; label=&quot;Title&quot; field=&quot;title&quot; mode=&quot;raw&quot;}{column}

    {column label=&quot;Modified&quot; sort=&quot;modification_date_major&quot; field=&quot;modification_date_major&quot;}

{OUTPUT}

{FORMAT(name=&quot;title&quot;)}{display name=title format=&quot;objectlink&quot;}{FORMAT}

{FORMAT(name=&quot;modification_date_major&quot;)}{display name=modification_date_major format=&quot;date&quot;}{FORMAT}

{LIST}

---

#### CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023

 **Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke** 

Stroke . 2023 Feb 10. [doi: 10.1161/STROKEAHA.122.040883](https://doi.org/10.1161/STROKEAHA.122.040883)  **[&lt;i class=&quot;fas fa-file-pdf&quot; style=&quot;margin-right: 0.3em;&quot;&gt;&lt;/i&gt;PDF of Preprint](/attachments/d3.mock.pdf)** 

Yuhan Xie # 1, Julián N Acosta # 2, Yixuan Ye 3, Zachariah S Demarais 4, Carolyn J Conlon 4, Ming Chen 1, Hongyu Zhao 1 3, Guido J Falcone 2

&lt;img src=&quot;/attachments/d3.mock.jpg&quot; alt=&quot;image&quot; width=&quot;800&quot;&gt;

Background: Ischemic stroke (IS) is a highly heritable trait, and genome-wide association studies have identified several commonly occurring susceptibility risk loci for this condition. However, there are limited data on the contribution of rare genetic variation to IS.

Methods: We conducted an exome-wide study using whole-exome sequencing data from  **152 058 UK Biobank participants** , including 1777 IS cases. We performed single-variant analyses for rare variants and gene-based analyses for loss-of-function and deleterious missense rare variants. We validated these results through

* (1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, 

* (2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and

* (3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.

Results: In single-variant association analyses, none of the evaluated variants were associated with IS at genome-wide significance levels (P&lt;5×10-8). In the gene-based analysis focused on loss-of-function and deleterious missense variants, rare genetic variation at CYP2R1 was significantly associated with IS risk (P=2.6×10-6), exceeding the Bonferroni-corrected threshold for 16 074 tests (P&lt;3.1×10-6). Validations analyses indicated that  **CYP2R1**  was associated with IS risk in MEGASTROKE (gene-based test, P=0.003), with IS recurrence in the VISP trial (gene-based test, P=0.001) and with neuroimaging traits (white matter hyperintensity, mean diffusivity, and fractional anisotropy) in the UK Biobank neuroimaging study (all gene-based tests, P&lt;0.05).

Conclusions: Because  **CYP2R1**  plays an important role in vitamin D metabolism and existing observational evidence suggests an association between vitamin D levels and cerebrovascular disease, our results support a role of this pathway in the occurrence of IS.

&lt;!-- ~tc~ (alias(CYP2R1 gene reduces Vitamin D - many studies)) ~/tc~ --&gt;


AST Structure:
-------------
├── BoldNode
│   full_match: `__People with a poor CYP2R1 genes are poor Vitamin D responders%%% ~hs~~hs~ Possible solution: take larger doses of Vitamin D__`
│   inner_content: `People with a poor CYP2R1 genes are poor Vitamin D responders%%% ~hs~~hs~ Possible solution: take larger doses of Vitamin D`
│   children:
│   ├── TextNode
│   │   full_match: `People with a poor CYP2R1 genes are poor Vitamin D responders%%% `
│   │   inner_content: `People with a poor CYP2R1 genes are poor Vitamin D responders%%% `
│   ├── HorizontalSpaceNode
│   │   full_match: `~hs~~hs~`
│   │   inner_content: ``
│   │   repetitions: `2`
│   ├── TextNode
│   │   full_match: ` Possible solution: take larger doses of Vitamin D`
│   │   inner_content: ` Possible solution: take larger doses of Vitamin D`├── TextNode
│   full_match: ` \n\n`
│   inner_content: ` \n\n`├── DivNode
│   full_match: `{DIV(class=&quot;lefth4&quot;)}{maketoc}{DIV}`
│   inner_content: `{maketoc}`
│   attrs_dict:
│   │   raw_content: (class=&quot;lefth4&quot;)
│   │   class: lefth4
│   children:
│   ├── MakeTocNode
│   │   full_match: `{maketoc}`
│   │   inner_content: ``
│   │   attrs_dict:
│   │   │   raw_content: ├── TextNode
│   full_match: ` `
│   inner_content: ` `├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!VitaminDWiki - ((Genetics)) chart shows the vitamin D genes`
│   inner_content: `VitaminDWiki - ((Genetics)) chart shows the vitamin D genes`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `VitaminDWiki - `
│   │   inner_content: `VitaminDWiki - `
│   ├── LocalLinkNode
│   │   full_match: `((Genetics))`
│   │   inner_content: `Genetics`
│   │   page: `Genetics`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Genetics`
│   │   │   inner_content: `Genetics`
│   ├── TextNode
│   │   full_match: ` chart shows the vitamin D genes`
│   │   inner_content: ` chart shows the vitamin D genes`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ImgNode
│   full_match: `{img type=&quot;fileId&quot; fileId=&quot;3239&quot; width=&quot;300&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content:  type=&quot;fileId&quot; fileId=&quot;3239&quot; width=&quot;300&quot;
│   │   type: fileId
│   │   fileId: 3239
│   │   width: 300├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!Genes and other reasons for poor blood response to Vitamin D supplementation`
│   inner_content: `Genes and other reasons for poor blood response to Vitamin D supplementation`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `Genes and other reasons for poor blood response to Vitamin D supplementation`
│   │   inner_content: `Genes and other reasons for poor blood response to Vitamin D supplementation`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022))`
│   inner_content: `((Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022))`
│   │   inner_content: `Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022`
│   │   page: `Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022`
│   │   │   inner_content: `Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Reasons for low response to vitamin D))`
│   inner_content: `((Reasons for low response to vitamin D))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Reasons for low response to vitamin D))`
│   │   inner_content: `Reasons for low response to vitamin D`
│   │   page: `Reasons for low response to vitamin D`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Reasons for low response to vitamin D`
│   │   │   inner_content: `Reasons for low response to vitamin D`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Autistics have half of the response to Vitamin D – RCT Oct 2018))`
│   inner_content: `((Autistics have half of the response to Vitamin D – RCT Oct 2018))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Autistics have half of the response to Vitamin D – RCT Oct 2018))`
│   │   inner_content: `Autistics have half of the response to Vitamin D – RCT Oct 2018`
│   │   page: `Autistics have half of the response to Vitamin D – RCT Oct 2018`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Autistics have half of the response to Vitamin D – RCT Oct 2018`
│   │   │   inner_content: `Autistics have half of the response to Vitamin D – RCT Oct 2018`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019))`
│   inner_content: `((Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019))`
│   │   inner_content: `Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019`
│   │   page: `Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019`
│   │   │   inner_content: `Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019))`
│   inner_content: `((Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019))`
│   │   inner_content: `Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019`
│   │   page: `Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019`
│   │   │   inner_content: `Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019))`
│   inner_content: `((Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019))`
│   │   inner_content: `Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019`
│   │   page: `Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019`
│   │   │   inner_content: `Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022))`
│   inner_content: `((Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022))`
│   │   inner_content: `Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022`
│   │   page: `Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022`
│   │   │   inner_content: `Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022`├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!24+ VitaminDWiki pages with CYP2R1 in title`
│   inner_content: `24+ VitaminDWiki pages with CYP2R1 in title`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `24+ VitaminDWiki pages with CYP2R1 in title`
│   │   inner_content: `24+ VitaminDWiki pages with CYP2R1 in title`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── EmphasisNode
│   full_match: `&#39;&#39;This list is automatically updated&#39;&#39;`
│   inner_content: `This list is automatically updated`
│   children:
│   ├── TextNode
│   │   full_match: `This list is automatically updated`
│   │   inner_content: `This list is automatically updated`├── TextNode
│   full_match: `\n{LIST()}\n{list max=&quot;50&quot;}\n`
│   inner_content: `\n{LIST()}\n{list max=&quot;50&quot;}\n`├── FilterNode
│   full_match: `{filter field=&quot;title&quot; content=&quot;CYP2R1&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content: field=&quot;title&quot; content=&quot;CYP2R1&quot;
│   │   field: title
│   │   content: CYP2R1├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── FilterNode
│   full_match: `{filter type=&quot;wiki page&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content: type=&quot;wiki page&quot;
│   │   type: wiki page├── TextNode
│   full_match: `\n{sort mode=&quot;modification_date_major_desc&quot;}\n{OUTPUT(template=&quot;table&quot;)}\n   {tableparams allowtableexpansion=&quot;n&quot; shownbitems=&quot;y&quot; title=&quot;&quot;}\n    {column sort=&quot;title&quot; label=&quot;Title&quot; field=&quot;title&quot; mode=&quot;raw&quot;}{column}\n    {column label=&quot;Modified&quot; sort=&quot;modification_date_major&quot; field=&quot;modification_date_major&quot;}\n{OUTPUT}\n{FORMAT(name=&quot;title&quot;)}{display name=title format=&quot;objectlink&quot;}{FORMAT}\n{FORMAT(name=&quot;modification_date_major&quot;)}{display name=modification_date_major format=&quot;date&quot;}{FORMAT}\n{LIST}`
│   inner_content: `\n{sort mode=&quot;modification_date_major_desc&quot;}\n{OUTPUT(template=&quot;table&quot;)}\n   {tableparams allowtableexpansion=&quot;n&quot; shownbitems=&quot;y&quot; title=&quot;&quot;}\n    {column sort=&quot;title&quot; label=&quot;Title&quot; field=&quot;title&quot; mode=&quot;raw&quot;}{column}\n    {column label=&quot;Modified&quot; sort=&quot;modification_date_major&quot; field=&quot;modification_date_major&quot;}\n{OUTPUT}\n{FORMAT(name=&quot;title&quot;)}{display name=title format=&quot;objectlink&quot;}{FORMAT}\n{FORMAT(name=&quot;modification_date_major&quot;)}{display name=modification_date_major format=&quot;date&quot;}{FORMAT}\n{LIST}`├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023`
│   inner_content: `CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023`
│   │   inner_content: `CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── BoldNode
│   full_match: `__Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke__`
│   inner_content: `Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke`
│   children:
│   ├── TextNode
│   │   full_match: `Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke`
│   │   inner_content: `Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke`├── TextNode
│   full_match: `\nStroke . 2023 Feb 10. `
│   inner_content: `\nStroke . 2023 Feb 10. `├── DoiLinkNode
│   full_match: `doi: 10.1161/STROKEAHA.122.040883`
│   inner_content: `10.1161/STROKEAHA.122.040883`├── TextNode
│   full_match: ` `
│   inner_content: ` `├── BoldNode
│   full_match: `__{ATTACH(inline=&quot;1&quot; id=&quot;19196&quot;)}PDF of Preprint{ATTACH}__`
│   inner_content: `{ATTACH(inline=&quot;1&quot; id=&quot;19196&quot;)}PDF of Preprint{ATTACH}`
│   children:
│   ├── AttachNode
│   │   full_match: `{ATTACH(inline=&quot;1&quot; id=&quot;19196&quot;)}PDF of Preprint{ATTACH}`
│   │   inner_content: `PDF of Preprint`
│   │   attrs_dict:
│   │   │   raw_content: (inline=&quot;1&quot; id=&quot;19196&quot;)
│   │   │   inline: 1
│   │   │   id: 19196
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `PDF of Preprint`
│   │   │   inner_content: `PDF of Preprint`├── TextNode
│   full_match: `\nYuhan Xie # 1, Julián N Acosta # 2, Yixuan Ye 3, Zachariah S Demarais 4, Carolyn J Conlon 4, Ming Chen 1, Hongyu Zhao 1 3, Guido J Falcone 2\n\n`
│   inner_content: `\nYuhan Xie # 1, Julián N Acosta # 2, Yixuan Ye 3, Zachariah S Demarais 4, Carolyn J Conlon 4, Ming Chen 1, Hongyu Zhao 1 3, Guido J Falcone 2\n\n`├── ImgNode
│   full_match: `{img type=&quot;attId&quot; attId=&quot;19196&quot; width=&quot;800&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content:  type=&quot;attId&quot; attId=&quot;19196&quot; width=&quot;800&quot;
│   │   type: attId
│   │   attId: 19196
│   │   width: 800├── TextNode
│   full_match: `\n\nBackground: Ischemic stroke (IS) is a highly heritable trait, and genome-wide association studies have identified several commonly occurring susceptibility risk loci for this condition. However, there are limited data on the contribution of rare genetic variation to IS.\n\nMethods: We conducted an exome-wide study using whole-exome sequencing data from `
│   inner_content: `\n\nBackground: Ischemic stroke (IS) is a highly heritable trait, and genome-wide association studies have identified several commonly occurring susceptibility risk loci for this condition. However, there are limited data on the contribution of rare genetic variation to IS.\n\nMethods: We conducted an exome-wide study using whole-exome sequencing data from `├── BoldNode
│   full_match: `__152 058 UK Biobank participants__`
│   inner_content: `152 058 UK Biobank participants`
│   children:
│   ├── TextNode
│   │   full_match: `152 058 UK Biobank participants`
│   │   inner_content: `152 058 UK Biobank participants`├── TextNode
│   full_match: `, including 1777 IS cases. We performed single-variant analyses for rare variants and gene-based analyses for loss-of-function and deleterious missense rare variants. We validated these results through\n`
│   inner_content: `, including 1777 IS cases. We performed single-variant analyses for rare variants and gene-based analyses for loss-of-function and deleterious missense rare variants. We validated these results through\n`├── ListItemNode
│   full_match: `*(1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, `
│   inner_content: `(1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, `
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `(1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, `
│   │   inner_content: `(1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, `├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*(2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and`
│   inner_content: `(2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `(2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and`
│   │   inner_content: `(2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*(3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.`
│   inner_content: `(3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `(3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.`
│   │   inner_content: `(3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.`├── TextNode
│   full_match: `\nResults: In single-variant association analyses, none of the evaluated variants were associated with IS at genome-wide significance levels (P&lt;5×10-8). In the gene-based analysis focused on loss-of-function and deleterious missense variants, rare genetic variation at CYP2R1 was significantly associated with IS risk (P=2.6×10-6), exceeding the Bonferroni-corrected threshold for 16 074 tests (P&lt;3.1×10-6). Validations analyses indicated that `
│   inner_content: `\nResults: In single-variant association analyses, none of the evaluated variants were associated with IS at genome-wide significance levels (P&lt;5×10-8). In the gene-based analysis focused on loss-of-function and deleterious missense variants, rare genetic variation at CYP2R1 was significantly associated with IS risk (P=2.6×10-6), exceeding the Bonferroni-corrected threshold for 16 074 tests (P&lt;3.1×10-6). Validations analyses indicated that `├── BoldNode
│   full_match: `__CYP2R1__`
│   inner_content: `CYP2R1`
│   children:
│   ├── TextNode
│   │   full_match: `CYP2R1`
│   │   inner_content: `CYP2R1`├── TextNode
│   full_match: ` was associated with IS risk in MEGASTROKE (gene-based test, P=0.003), with IS recurrence in the VISP trial (gene-based test, P=0.001) and with neuroimaging traits (white matter hyperintensity, mean diffusivity, and fractional anisotropy) in the UK Biobank neuroimaging study (all gene-based tests, P&lt;0.05).\n\nConclusions: Because `
│   inner_content: ` was associated with IS risk in MEGASTROKE (gene-based test, P=0.003), with IS recurrence in the VISP trial (gene-based test, P=0.001) and with neuroimaging traits (white matter hyperintensity, mean diffusivity, and fractional anisotropy) in the UK Biobank neuroimaging study (all gene-based tests, P&lt;0.05).\n\nConclusions: Because `├── BoldNode
│   full_match: `__CYP2R1__`
│   inner_content: `CYP2R1`
│   children:
│   ├── TextNode
│   │   full_match: `CYP2R1`
│   │   inner_content: `CYP2R1`├── TextNode
│   full_match: ` plays an important role in vitamin D metabolism and existing observational evidence suggests an association between vitamin D levels and cerebrovascular disease, our results support a role of this pathway in the occurrence of IS.\n\n`
│   inner_content: ` plays an important role in vitamin D metabolism and existing observational evidence suggests an association between vitamin D levels and cerebrovascular disease, our results support a role of this pathway in the occurrence of IS.\n\n`├── TikiCommentNode
│   full_match: `~tc~ (alias(CYP2R1 gene reduces Vitamin D - many studies)) ~/tc~`
│   inner_content: ` (alias(CYP2R1 gene reduces Vitamin D - many studies)) `
│   children:
│   ├── TextNode
│   │   full_match: ` (alias(CYP2R1 gene reduces Vitamin D - many studies)) `
│   │   inner_content: ` (alias(CYP2R1 gene reduces Vitamin D - many studies)) `

Original Tiki:
-------------
__People with a poor CYP2R1 genes are poor Vitamin D responders%%% ~hs~~hs~ Possible solution: take larger doses of Vitamin D__ 

{DIV(class=&quot;lefth4&quot;)}{maketoc}{DIV} 
---
!!!!VitaminDWiki - ((Genetics)) chart shows the vitamin D genes
{img type=&quot;fileId&quot; fileId=&quot;3239&quot; width=&quot;300&quot;}
---
!!!!Genes and other reasons for poor blood response to Vitamin D supplementation
*((Poor or no response to vitamin D was associated with poor genes (cystic fibrosis, 4 genes) Sept 2022))
*((Reasons for low response to vitamin D))
*((Autistics have half of the response to Vitamin D – RCT Oct 2018))
*((Poor CYP2R1 gene reduces blood response to Vitamin D supplementation – Aug 2019))
*((Poor Vitamin D response 4X more likely if poor Vitamin D binding proteins - July 2019))
*((Vitamin D Nutrigenomics - High, Medium, and Low Responders - March 2019))
*((Vitamin D non-responders may have one or more poor genes: GC, LIPC, CYP24A1, and PDE3B – Oct 2022))
---
!!!!24+ VitaminDWiki pages with CYP2R1 in title
&#39;&#39;This list is automatically updated&#39;&#39;
{LIST()}
{list max=&quot;50&quot;}
{filter field=&quot;title&quot; content=&quot;CYP2R1&quot;}
{filter type=&quot;wiki page&quot;}
{sort mode=&quot;modification_date_major_desc&quot;}
{OUTPUT(template=&quot;table&quot;)}
   {tableparams allowtableexpansion=&quot;n&quot; shownbitems=&quot;y&quot; title=&quot;&quot;}
    {column sort=&quot;title&quot; label=&quot;Title&quot; field=&quot;title&quot; mode=&quot;raw&quot;}{column}
    {column label=&quot;Modified&quot; sort=&quot;modification_date_major&quot; field=&quot;modification_date_major&quot;}
{OUTPUT}
{FORMAT(name=&quot;title&quot;)}{display name=title format=&quot;objectlink&quot;}{FORMAT}
{FORMAT(name=&quot;modification_date_major&quot;)}{display name=modification_date_major format=&quot;date&quot;}{FORMAT}
{LIST}
---
!!!!CYP2R1 was the gene most associated with increased risk of Ischemic Stroke - Feb 2023
__Whole-Exome Sequencing Analyses Support a Role of Vitamin D Metabolism in Ischemic Stroke__
Stroke . 2023 Feb 10. doi: 10.1161/STROKEAHA.122.040883 __{ATTACH(inline=&quot;1&quot; id=&quot;19196&quot;)}PDF of Preprint{ATTACH}__
Yuhan Xie # 1, Julián N Acosta # 2, Yixuan Ye 3, Zachariah S Demarais 4, Carolyn J Conlon 4, Ming Chen 1, Hongyu Zhao 1 3, Guido J Falcone 2

{img type=&quot;attId&quot; attId=&quot;19196&quot; width=&quot;800&quot;}

Background: Ischemic stroke (IS) is a highly heritable trait, and genome-wide association studies have identified several commonly occurring susceptibility risk loci for this condition. However, there are limited data on the contribution of rare genetic variation to IS.

Methods: We conducted an exome-wide study using whole-exome sequencing data from __152 058 UK Biobank participants__, including 1777 IS cases. We performed single-variant analyses for rare variants and gene-based analyses for loss-of-function and deleterious missense rare variants. We validated these results through
*(1) gene-based testing using summary statistics from MEGASTROKE-a genome-wide association study of IS that included 67 162 IS cases and 454 450 controls, 
*(2) gene-based testing using individual-level data from 1706 IS survivors, including 142 recurrent IS cases, enrolled in the VISP trial (Vitamin Intervention for Stroke Prevention); and
*(3) gene-based testing against neuroimaging phenotypes related to cerebrovascular disease using summary-level data from 42,310 UK Biobank participants with available magnetic resonance imaging data.
Results: In single-variant association analyses, none of the evaluated variants were associated with IS at genome-wide significance levels (P&lt;5×10-8). In the gene-based analysis focused on loss-of-function and deleterious missense variants, rare genetic variation at CYP2R1 was significantly associated with IS risk (P=2.6×10-6), exceeding the Bonferroni-corrected threshold for 16 074 tests (P&lt;3.1×10-6). Validations analyses indicated that __CYP2R1__ was associated with IS risk in MEGASTROKE (gene-based test, P=0.003), with IS recurrence in the VISP trial (gene-based test, P=0.001) and with neuroimaging traits (white matter hyperintensity, mean diffusivity, and fractional anisotropy) in the UK Biobank neuroimaging study (all gene-based tests, P&lt;0.05).

Conclusions: Because __CYP2R1__ plays an important role in vitamin D metabolism and existing observational evidence suggests an association between vitamin D levels and cerebrovascular disease, our results support a role of this pathway in the occurrence of IS.

~tc~ (alias(CYP2R1 gene reduces Vitamin D - many studies)) ~/tc~
</code>
</pre>
