---
title: "People with Multiple Sclerosis have blunted responses to Vitamin D supplementation"
slug: "people-with-multiple-sclerosis-have-blunted-responses-to-vitamin-d-supplementation"
aliases:
  - "/People+with+Multiple+Sclerosis+have+blunted+responses+to+Vitamin+D+supplementation+-+Jan+2024"
  - "/tiki-index.php?page_id=14976"
date: 2024-01-17
categories:
  - "Multiple Sclerosis"
  - "Genetics"
---



<!-- {maketoc Title=""} -->

---

#### Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis

Scientific Reports Vol 14, Article number: 1436 (2024)  https://doi.org/10.1038/s41598-024-51779-0

Wei Z. Yeh, Rodney Lea, Jim Stankovich, Sandeep Sampangi, Louise Laverick, Anneke Van der Walt, Vilija Jokubaitis, Melissa Gresle & Helmut Butzkueven 

Vitamin D deficiency is a risk factor for developing multiple sclerosis (MS). However, the immune effects of vitamin D in people with MS are not well understood. We analyzed transcriptomic datasets generated by RNA sequencing of immune cell subsets (CD4+, CD8+ T cells, B cells, monocytes) from 33 healthy controls and 33 untreated MS cases. We utilized a traditional bioinformatic pipeline and weighted gene co-expression network analysis (WGCNA) to determine genes and pathways correlated with endogenous vitamin D. In controls, CD4+ and CD8+ T cells had 1079 and 1188 genes, respectively, whose expressions were correlated with plasma 25-hydroxyvitamin D level (P < 0.05). Functional enrichment analysis identified association with TNF-alpha and MAPK signaling. In CD4+ T cells of controls, vitamin D level was associated with expression levels of several genes proximal to multiple sclerosis risk loci (P = 0.01). Genes differentially associated with endogenous vitamin D by case–control status were enriched in TNF-alpha signaling via NF-κB. WGCNA suggested a blunted response to vitamin D in cases relative to controls. Collectively, our findings provide further evidence for the immune effects of vitamin D, and demonstrate a differential immune response to vitamin D in cases relative to controls, highlighting a possible mechanism contributing to MS pathophysiology.

 **[<i class="fas fa-file-pdf" style="margin-right: 0.3em;"></i>Download the PDF from VitaminDWiki ](/attachments/d3.mock.pdf)** 

---

##### 82 References

1. Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).

1. Veldman, C. M., Cantorna, M. T. & DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).

1. Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).

1. Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).

1. Liu, P. T., Stenger, S., Tang, D. H. & Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).

1. Penna, G. & Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).

1. Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).

1. Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).

1. Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).

1. Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. & Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).

1. da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).

1. Kickler, K., Ni Choileain, S., Williams, A., Richards, A. & Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).

1. Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).

1. Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).

1. Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).

1. International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).

1. Zhang, B. & Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).

1. Langfelder, P. & Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).

1. Kriegler, M., Perez, C., DeFay, K., Albert, I. & Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).

1. Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).

1. Park, K. M. & Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).

1. Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).

1. Sharief, M. K. & Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).

1. Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).

1. Huang, W.-X., Huang, P., Link, H. & Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).

1. van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).

1. Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).

1. Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).

1. Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).

1. Peterson, C. A. & Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).

1. Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).

1. Song, G. G., Bae, S.-C. & Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).

1. Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn's disease. Gastroenterology 142, 482-489 (2012).

1. Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).

1. Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).

1. Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).

1. Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).

1. von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).

1. Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).

1. Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. & Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).

1. Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).

1. Ding, C., Wilding, J. P H. & Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).

1. Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).

1. Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).

1. Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).

1. Matilainen, J. M., Räsänen, A., Gynther, P. & Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).

1. Alroy, I., Towers, T. L. & Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).

1. Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).

1. Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).

1. Müller, K., 0dum, N. & Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).

1. Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).

1. Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).

1. Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).

1. Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. & Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).

1. Lu, M., Shi, H., Taylor, B. V. & Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).

1. Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).

1. Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).

1. Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).

1. Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).

1. Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).

1. Bray, N. L., Pimentel, H., Melsted, P. & Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).

1. R Core Team. R: A Language and Environment for Statistical Computing (2019).

1. Soneson, C., Love, M. I. & Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).

1. Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).

1. Risso, D., Ngai, J., Speed, T. P. & Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).

1. Eisenberg, E. & Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).

1. Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).

1. Neme, A., Seuter, S. & Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).

1. Shirvani, A., Kalajian, T. A., Song, A. & Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).

1. Gandolfo, L. C. & Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).

1. Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).

1. Durinck, S., Spellman, P. T., Birney, E. & Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).

1. Lun, A. T. L., Chen, Y. & Smyth, G. K. It's DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. & Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.

1. Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531

1. Langfelder, P. & Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).

1. Benjamini, Y. & Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).

1. Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).

1. Kanehisa, M. & Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).

1. Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).

1. Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.

1. Chen, J., Bardes, E. E., Aronow, B. J. & Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).

1. Gu, Z., Eils, R. & Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).

---

#### 
<!-- {SQL(db=>vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) > 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = "wiki page"{SQL} -->
    Items in both categories MS and Genetics:

  

---

#### 9 related items in VitaminDWiki

* <a href="/posts/epstein-barr-virus-increases-risk-of-multiple-sclerosis-by-32x" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022">Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022</a>

  * Many virus have learned to protect themselves by deactivating the Vitamin D Receptor

  * Which results in less Vitamin D getting to the cells

* <a href="/posts/epigenetics-of-multiple-sclerosis" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Epigenetics of Multiple Sclerosis – March 2014">Epigenetics of Multiple Sclerosis – March 2014</a>

* <a href="/posts/resveratrol-treats-multiple-sclerosis-and-other-autoimmune-diseases-many-studies" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies">Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies</a>

* <a href="/posts/gene-variants-can-reduce-vitamin-d-response-by-17x-14000-iu-daily-multiple-sclerosis" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021">Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021</a>

* <a href="/posts/uk-people-with-multiple-sclerosis-are-3x-more-likely-to-take-vitamin-d" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020">UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020</a>

* <a href="/posts/multiple-sclerosis-much-more-likely-if-poor-vitamin-d-receptor-several-studies" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies">Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies</a>

* <a href="/posts/7-ng-less-response-by-multiple-sclerosis-patients-to-5000-iu-of-vitamin-d" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: 7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015">7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015</a>

* <a href="/posts/gene-variants-can-reduce-vitamin-d-response-by-17x-14000-iu-daily-multiple-sclerosis" style="color: red; text-decoration: underline;" title="This post/category does not exist yet: Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021">Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021</a>

---

#### Several Health problems have a blunted response to Vitamin D supplementation

* [Autism](https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342) 2018

* [Obesity](https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491)  Jan 2023

---

#### 
<!-- {SQL(db=>vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) > 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = "wiki page"{SQL} -->
    Items in both categories MS and Vitamin D Receptor:

  

---

#### Strange that studies have not tried activating the VDR in people with MS

---

#### VitaminDWiki - [Vitamin D Receptor](/categories/vitamin-d-receptor) activation can be increased in many ways

<!-- ~tc~ (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) ~/tc~ -->

<!-- ~tc~ (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) ~/tc~ -->


<pre style="background-color: #e0e0e0;">
<code class="language-text">
Markdown:
--------

&lt;!-- {maketoc Title=&quot;&quot;} --&gt;

---

#### Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis

Scientific Reports Vol 14, Article number: 1436 (2024)  https://doi.org/10.1038/s41598-024-51779-0

Wei Z. Yeh, Rodney Lea, Jim Stankovich, Sandeep Sampangi, Louise Laverick, Anneke Van der Walt, Vilija Jokubaitis, Melissa Gresle &amp; Helmut Butzkueven 

Vitamin D deficiency is a risk factor for developing multiple sclerosis (MS). However, the immune effects of vitamin D in people with MS are not well understood. We analyzed transcriptomic datasets generated by RNA sequencing of immune cell subsets (CD4+, CD8+ T cells, B cells, monocytes) from 33 healthy controls and 33 untreated MS cases. We utilized a traditional bioinformatic pipeline and weighted gene co-expression network analysis (WGCNA) to determine genes and pathways correlated with endogenous vitamin D. In controls, CD4+ and CD8+ T cells had 1079 and 1188 genes, respectively, whose expressions were correlated with plasma 25-hydroxyvitamin D level (P &lt; 0.05). Functional enrichment analysis identified association with TNF-alpha and MAPK signaling. In CD4+ T cells of controls, vitamin D level was associated with expression levels of several genes proximal to multiple sclerosis risk loci (P = 0.01). Genes differentially associated with endogenous vitamin D by case–control status were enriched in TNF-alpha signaling via NF-κB. WGCNA suggested a blunted response to vitamin D in cases relative to controls. Collectively, our findings provide further evidence for the immune effects of vitamin D, and demonstrate a differential immune response to vitamin D in cases relative to controls, highlighting a possible mechanism contributing to MS pathophysiology.

 **[&lt;i class=&quot;fas fa-file-pdf&quot; style=&quot;margin-right: 0.3em;&quot;&gt;&lt;/i&gt;Download the PDF from VitaminDWiki ](/attachments/d3.mock.pdf)** 

---

##### 82 References

1. Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).

1. Veldman, C. M., Cantorna, M. T. &amp; DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).

1. Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).

1. Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).

1. Liu, P. T., Stenger, S., Tang, D. H. &amp; Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).

1. Penna, G. &amp; Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).

1. Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).

1. Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).

1. Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).

1. Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. &amp; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).

1. da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).

1. Kickler, K., Ni Choileain, S., Williams, A., Richards, A. &amp; Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).

1. Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).

1. Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).

1. Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).

1. International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).

1. Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).

1. Langfelder, P. &amp; Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).

1. Kriegler, M., Perez, C., DeFay, K., Albert, I. &amp; Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).

1. Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).

1. Park, K. M. &amp; Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).

1. Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).

1. Sharief, M. K. &amp; Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).

1. Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).

1. Huang, W.-X., Huang, P., Link, H. &amp; Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).

1. van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).

1. Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).

1. Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).

1. Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).

1. Peterson, C. A. &amp; Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).

1. Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).

1. Song, G. G., Bae, S.-C. &amp; Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).

1. Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn&#39;s disease. Gastroenterology 142, 482-489 (2012).

1. Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).

1. Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).

1. Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).

1. Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).

1. von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).

1. Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).

1. Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. &amp; Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).

1. Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).

1. Ding, C., Wilding, J. P H. &amp; Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).

1. Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).

1. Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).

1. Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).

1. Matilainen, J. M., Räsänen, A., Gynther, P. &amp; Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).

1. Alroy, I., Towers, T. L. &amp; Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).

1. Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).

1. Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).

1. Müller, K., 0dum, N. &amp; Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).

1. Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).

1. Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).

1. Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).

1. Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. &amp; Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).

1. Lu, M., Shi, H., Taylor, B. V. &amp; Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).

1. Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).

1. Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).

1. Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).

1. Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).

1. Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).

1. Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).

1. R Core Team. R: A Language and Environment for Statistical Computing (2019).

1. Soneson, C., Love, M. I. &amp; Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).

1. Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).

1. Risso, D., Ngai, J., Speed, T. P. &amp; Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).

1. Eisenberg, E. &amp; Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).

1. Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).

1. Neme, A., Seuter, S. &amp; Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).

1. Shirvani, A., Kalajian, T. A., Song, A. &amp; Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).

1. Gandolfo, L. C. &amp; Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).

1. Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).

1. Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).

1. Lun, A. T. L., Chen, Y. &amp; Smyth, G. K. It&#39;s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. &amp; Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.

1. Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531

1. Langfelder, P. &amp; Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).

1. Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).

1. Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).

1. Kanehisa, M. &amp; Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).

1. Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).

1. Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.

1. Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).

1. Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).

---

#### 
&lt;!-- {SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL} --&gt;
    Items in both categories MS and Genetics:

  

---

#### 9 related items in VitaminDWiki

* &lt;a href=&quot;/posts/epstein-barr-virus-increases-risk-of-multiple-sclerosis-by-32x&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022&quot;&gt;Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022&lt;/a&gt;

  * Many virus have learned to protect themselves by deactivating the Vitamin D Receptor

  * Which results in less Vitamin D getting to the cells

* &lt;a href=&quot;/posts/epigenetics-of-multiple-sclerosis&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Epigenetics of Multiple Sclerosis – March 2014&quot;&gt;Epigenetics of Multiple Sclerosis – March 2014&lt;/a&gt;

* &lt;a href=&quot;/posts/resveratrol-treats-multiple-sclerosis-and-other-autoimmune-diseases-many-studies&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies&quot;&gt;Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies&lt;/a&gt;

* &lt;a href=&quot;/posts/gene-variants-can-reduce-vitamin-d-response-by-17x-14000-iu-daily-multiple-sclerosis&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021&quot;&gt;Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021&lt;/a&gt;

* &lt;a href=&quot;/posts/uk-people-with-multiple-sclerosis-are-3x-more-likely-to-take-vitamin-d&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020&quot;&gt;UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020&lt;/a&gt;

* &lt;a href=&quot;/posts/multiple-sclerosis-much-more-likely-if-poor-vitamin-d-receptor-several-studies&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies&quot;&gt;Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies&lt;/a&gt;

* &lt;a href=&quot;/posts/7-ng-less-response-by-multiple-sclerosis-patients-to-5000-iu-of-vitamin-d&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: 7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015&quot;&gt;7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015&lt;/a&gt;

* &lt;a href=&quot;/posts/gene-variants-can-reduce-vitamin-d-response-by-17x-14000-iu-daily-multiple-sclerosis&quot; style=&quot;color: red; text-decoration: underline;&quot; title=&quot;This post/category does not exist yet: Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021&quot;&gt;Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021&lt;/a&gt;

---

#### Several Health problems have a blunted response to Vitamin D supplementation

* [Autism](https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342) 2018

* [Obesity](https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491)  Jan 2023

---

#### 
&lt;!-- {SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL} --&gt;
    Items in both categories MS and Vitamin D Receptor:

  

---

#### Strange that studies have not tried activating the VDR in people with MS

---

#### VitaminDWiki - [Vitamin D Receptor](/categories/vitamin-d-receptor) activation can be increased in many ways

&lt;!-- ~tc~ (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) ~/tc~ --&gt;

&lt;!-- ~tc~ (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) ~/tc~ --&gt;


AST Structure:
-------------
├── DivNode
│   full_match: `{DIV(class=&quot;lefth4&quot;)}{maketoc Title=&quot;&quot;}{DIV}`
│   inner_content: `{maketoc Title=&quot;&quot;}`
│   attrs_dict:
│   │   raw_content: (class=&quot;lefth4&quot;)
│   │   class: lefth4
│   children:
│   ├── MakeTocNode
│   │   full_match: `{maketoc Title=&quot;&quot;}`
│   │   inner_content: ``
│   │   attrs_dict:
│   │   │   raw_content:  Title=&quot;&quot;
│   │   │   Title: ├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis`
│   inner_content: `Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis`
│   │   inner_content: `Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis`├── TextNode
│   full_match: `\nScientific Reports Vol 14, Article number: 1436 (2024)  https://doi.org/10.1038/s41598-024-51779-0\nWei Z. Yeh, Rodney Lea, Jim Stankovich, Sandeep Sampangi, Louise Laverick, Anneke Van der Walt, Vilija Jokubaitis, Melissa Gresle &amp; Helmut Butzkueven \n\nVitamin D deficiency is a risk factor for developing multiple sclerosis (MS). However, the immune effects of vitamin D in people with MS are not well understood. We analyzed transcriptomic datasets generated by RNA sequencing of immune cell subsets (CD4+, CD8+ T cells, B cells, monocytes) from 33 healthy controls and 33 untreated MS cases. We utilized a traditional bioinformatic pipeline and weighted gene co-expression network analysis (WGCNA) to determine genes and pathways correlated with endogenous vitamin D. In controls, CD4+ and CD8+ T cells had 1079 and 1188 genes, respectively, whose expressions were correlated with plasma 25-hydroxyvitamin D level (P &lt; 0.05). Functional enrichment analysis identified association with TNF-alpha and MAPK signaling. In CD4+ T cells of controls, vitamin D level was associated with expression levels of several genes proximal to multiple sclerosis risk loci (P = 0.01). Genes differentially associated with endogenous vitamin D by case–control status were enriched in TNF-alpha signaling via NF-κB. WGCNA suggested a blunted response to vitamin D in cases relative to controls. Collectively, our findings provide further evidence for the immune effects of vitamin D, and demonstrate a differential immune response to vitamin D in cases relative to controls, highlighting a possible mechanism contributing to MS pathophysiology.\n`
│   inner_content: `\nScientific Reports Vol 14, Article number: 1436 (2024)  https://doi.org/10.1038/s41598-024-51779-0\nWei Z. Yeh, Rodney Lea, Jim Stankovich, Sandeep Sampangi, Louise Laverick, Anneke Van der Walt, Vilija Jokubaitis, Melissa Gresle &amp; Helmut Butzkueven \n\nVitamin D deficiency is a risk factor for developing multiple sclerosis (MS). However, the immune effects of vitamin D in people with MS are not well understood. We analyzed transcriptomic datasets generated by RNA sequencing of immune cell subsets (CD4+, CD8+ T cells, B cells, monocytes) from 33 healthy controls and 33 untreated MS cases. We utilized a traditional bioinformatic pipeline and weighted gene co-expression network analysis (WGCNA) to determine genes and pathways correlated with endogenous vitamin D. In controls, CD4+ and CD8+ T cells had 1079 and 1188 genes, respectively, whose expressions were correlated with plasma 25-hydroxyvitamin D level (P &lt; 0.05). Functional enrichment analysis identified association with TNF-alpha and MAPK signaling. In CD4+ T cells of controls, vitamin D level was associated with expression levels of several genes proximal to multiple sclerosis risk loci (P = 0.01). Genes differentially associated with endogenous vitamin D by case–control status were enriched in TNF-alpha signaling via NF-κB. WGCNA suggested a blunted response to vitamin D in cases relative to controls. Collectively, our findings provide further evidence for the immune effects of vitamin D, and demonstrate a differential immune response to vitamin D in cases relative to controls, highlighting a possible mechanism contributing to MS pathophysiology.\n`├── FontNode
│   full_match: `{FONT(size=&quot;20&quot;)}__{ATTACH(inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)}Download the PDF from VitaminDWiki {ATTACH}__{FONT}`
│   inner_content: `__{ATTACH(inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)}Download the PDF from VitaminDWiki {ATTACH}__`
│   attrs_dict:
│   │   raw_content: size=&quot;20&quot;
│   │   size: 20
│   children:
│   ├── BoldNode
│   │   full_match: `__{ATTACH(inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)}Download the PDF from VitaminDWiki {ATTACH}__`
│   │   inner_content: `{ATTACH(inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)}Download the PDF from VitaminDWiki {ATTACH}`
│   │   children:
│   │   ├── AttachNode
│   │   │   full_match: `{ATTACH(inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)}Download the PDF from VitaminDWiki {ATTACH}`
│   │   │   inner_content: `Download the PDF from VitaminDWiki `
│   │   │   attrs_dict:
│   │   │   │   raw_content: (inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)
│   │   │   │   inline: 1
│   │   │   │   id: 20669
│   │   │   │   icon: 1
│   │   │   children:
│   │   │   ├── TextNode
│   │   │   │   full_match: `Download the PDF from VitaminDWiki `
│   │   │   │   inner_content: `Download the PDF from VitaminDWiki `├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!!82 References`
│   inner_content: `82 References`
│   level: `5`
│   children:
│   ├── TextNode
│   │   full_match: `82 References`
│   │   inner_content: `82 References`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).`
│   inner_content: `Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).`
│   │   inner_content: `Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Veldman, C. M., Cantorna, M. T. &amp; DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).`
│   inner_content: `Veldman, C. M., Cantorna, M. T. &amp; DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Veldman, C. M., Cantorna, M. T. &amp; DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).`
│   │   inner_content: `Veldman, C. M., Cantorna, M. T. &amp; DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).`
│   inner_content: `Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).`
│   │   inner_content: `Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).`
│   inner_content: `Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).`
│   │   inner_content: `Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Liu, P. T., Stenger, S., Tang, D. H. &amp; Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).`
│   inner_content: `Liu, P. T., Stenger, S., Tang, D. H. &amp; Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Liu, P. T., Stenger, S., Tang, D. H. &amp; Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).`
│   │   inner_content: `Liu, P. T., Stenger, S., Tang, D. H. &amp; Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Penna, G. &amp; Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).`
│   inner_content: `Penna, G. &amp; Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Penna, G. &amp; Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).`
│   │   inner_content: `Penna, G. &amp; Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).`
│   inner_content: `Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).`
│   │   inner_content: `Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).`
│   inner_content: `Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).`
│   │   inner_content: `Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).`
│   inner_content: `Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).`
│   │   inner_content: `Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. &amp; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).`
│   inner_content: `Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. &amp; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. &amp; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).`
│   │   inner_content: `Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. &amp; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).`
│   inner_content: `da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).`
│   │   inner_content: `da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Kickler, K., Ni Choileain, S., Williams, A., Richards, A. &amp; Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).`
│   inner_content: `Kickler, K., Ni Choileain, S., Williams, A., Richards, A. &amp; Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Kickler, K., Ni Choileain, S., Williams, A., Richards, A. &amp; Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).`
│   │   inner_content: `Kickler, K., Ni Choileain, S., Williams, A., Richards, A. &amp; Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).`
│   inner_content: `Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).`
│   │   inner_content: `Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).`
│   inner_content: `Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).`
│   │   inner_content: `Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).`
│   inner_content: `Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).`
│   │   inner_content: `Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).`
│   inner_content: `International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).`
│   │   inner_content: `International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).`
│   inner_content: `Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).`
│   │   inner_content: `Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Langfelder, P. &amp; Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).`
│   inner_content: `Langfelder, P. &amp; Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Langfelder, P. &amp; Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).`
│   │   inner_content: `Langfelder, P. &amp; Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Kriegler, M., Perez, C., DeFay, K., Albert, I. &amp; Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).`
│   inner_content: `Kriegler, M., Perez, C., DeFay, K., Albert, I. &amp; Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Kriegler, M., Perez, C., DeFay, K., Albert, I. &amp; Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).`
│   │   inner_content: `Kriegler, M., Perez, C., DeFay, K., Albert, I. &amp; Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).`
│   inner_content: `Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).`
│   │   inner_content: `Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Park, K. M. &amp; Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).`
│   inner_content: `Park, K. M. &amp; Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Park, K. M. &amp; Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).`
│   │   inner_content: `Park, K. M. &amp; Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).`
│   inner_content: `Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).`
│   │   inner_content: `Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Sharief, M. K. &amp; Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).`
│   inner_content: `Sharief, M. K. &amp; Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Sharief, M. K. &amp; Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).`
│   │   inner_content: `Sharief, M. K. &amp; Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).`
│   inner_content: `Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).`
│   │   inner_content: `Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Huang, W.-X., Huang, P., Link, H. &amp; Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).`
│   inner_content: `Huang, W.-X., Huang, P., Link, H. &amp; Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Huang, W.-X., Huang, P., Link, H. &amp; Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).`
│   │   inner_content: `Huang, W.-X., Huang, P., Link, H. &amp; Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).`
│   inner_content: `van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).`
│   │   inner_content: `van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).`
│   inner_content: `Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).`
│   │   inner_content: `Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).`
│   inner_content: `Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).`
│   │   inner_content: `Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).`
│   inner_content: `Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).`
│   │   inner_content: `Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Peterson, C. A. &amp; Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).`
│   inner_content: `Peterson, C. A. &amp; Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Peterson, C. A. &amp; Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).`
│   │   inner_content: `Peterson, C. A. &amp; Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).`
│   inner_content: `Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).`
│   │   inner_content: `Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Song, G. G., Bae, S.-C. &amp; Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).`
│   inner_content: `Song, G. G., Bae, S.-C. &amp; Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Song, G. G., Bae, S.-C. &amp; Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).`
│   │   inner_content: `Song, G. G., Bae, S.-C. &amp; Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn&#39;s disease. Gastroenterology 142, 482-489 (2012).`
│   inner_content: `Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn&#39;s disease. Gastroenterology 142, 482-489 (2012).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn&#39;s disease. Gastroenterology 142, 482-489 (2012).`
│   │   inner_content: `Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn&#39;s disease. Gastroenterology 142, 482-489 (2012).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).`
│   inner_content: `Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).`
│   │   inner_content: `Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).`
│   inner_content: `Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).`
│   │   inner_content: `Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).`
│   inner_content: `Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).`
│   │   inner_content: `Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).`
│   inner_content: `Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).`
│   │   inner_content: `Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).`
│   inner_content: `von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).`
│   │   inner_content: `von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).`
│   inner_content: `Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).`
│   │   inner_content: `Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. &amp; Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).`
│   inner_content: `Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. &amp; Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. &amp; Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).`
│   │   inner_content: `Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. &amp; Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).`
│   inner_content: `Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).`
│   │   inner_content: `Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Ding, C., Wilding, J. P H. &amp; Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).`
│   inner_content: `Ding, C., Wilding, J. P H. &amp; Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Ding, C., Wilding, J. P H. &amp; Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).`
│   │   inner_content: `Ding, C., Wilding, J. P H. &amp; Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).`
│   inner_content: `Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).`
│   │   inner_content: `Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).`
│   inner_content: `Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).`
│   │   inner_content: `Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).`
│   inner_content: `Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).`
│   │   inner_content: `Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Matilainen, J. M., Räsänen, A., Gynther, P. &amp; Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).`
│   inner_content: `Matilainen, J. M., Räsänen, A., Gynther, P. &amp; Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Matilainen, J. M., Räsänen, A., Gynther, P. &amp; Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).`
│   │   inner_content: `Matilainen, J. M., Räsänen, A., Gynther, P. &amp; Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Alroy, I., Towers, T. L. &amp; Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).`
│   inner_content: `Alroy, I., Towers, T. L. &amp; Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Alroy, I., Towers, T. L. &amp; Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).`
│   │   inner_content: `Alroy, I., Towers, T. L. &amp; Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).`
│   inner_content: `Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).`
│   │   inner_content: `Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).`
│   inner_content: `Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).`
│   │   inner_content: `Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Müller, K., 0dum, N. &amp; Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).`
│   inner_content: `Müller, K., 0dum, N. &amp; Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Müller, K., 0dum, N. &amp; Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).`
│   │   inner_content: `Müller, K., 0dum, N. &amp; Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).`
│   inner_content: `Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).`
│   │   inner_content: `Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).`
│   inner_content: `Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).`
│   │   inner_content: `Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).`
│   inner_content: `Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).`
│   │   inner_content: `Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. &amp; Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).`
│   inner_content: `Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. &amp; Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. &amp; Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).`
│   │   inner_content: `Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. &amp; Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Lu, M., Shi, H., Taylor, B. V. &amp; Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).`
│   inner_content: `Lu, M., Shi, H., Taylor, B. V. &amp; Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Lu, M., Shi, H., Taylor, B. V. &amp; Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).`
│   │   inner_content: `Lu, M., Shi, H., Taylor, B. V. &amp; Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).`
│   inner_content: `Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).`
│   │   inner_content: `Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).`
│   inner_content: `Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).`
│   │   inner_content: `Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).`
│   inner_content: `Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).`
│   │   inner_content: `Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).`
│   inner_content: `Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).`
│   │   inner_content: `Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).`
│   inner_content: `Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).`
│   │   inner_content: `Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).`
│   inner_content: `Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).`
│   │   inner_content: `Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#R Core Team. R: A Language and Environment for Statistical Computing (2019).`
│   inner_content: `R Core Team. R: A Language and Environment for Statistical Computing (2019).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `R Core Team. R: A Language and Environment for Statistical Computing (2019).`
│   │   inner_content: `R Core Team. R: A Language and Environment for Statistical Computing (2019).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Soneson, C., Love, M. I. &amp; Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).`
│   inner_content: `Soneson, C., Love, M. I. &amp; Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Soneson, C., Love, M. I. &amp; Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).`
│   │   inner_content: `Soneson, C., Love, M. I. &amp; Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).`
│   inner_content: `Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).`
│   │   inner_content: `Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Risso, D., Ngai, J., Speed, T. P. &amp; Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).`
│   inner_content: `Risso, D., Ngai, J., Speed, T. P. &amp; Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Risso, D., Ngai, J., Speed, T. P. &amp; Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).`
│   │   inner_content: `Risso, D., Ngai, J., Speed, T. P. &amp; Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Eisenberg, E. &amp; Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).`
│   inner_content: `Eisenberg, E. &amp; Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Eisenberg, E. &amp; Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).`
│   │   inner_content: `Eisenberg, E. &amp; Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).`
│   inner_content: `Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).`
│   │   inner_content: `Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Neme, A., Seuter, S. &amp; Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).`
│   inner_content: `Neme, A., Seuter, S. &amp; Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Neme, A., Seuter, S. &amp; Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).`
│   │   inner_content: `Neme, A., Seuter, S. &amp; Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Shirvani, A., Kalajian, T. A., Song, A. &amp; Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).`
│   inner_content: `Shirvani, A., Kalajian, T. A., Song, A. &amp; Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Shirvani, A., Kalajian, T. A., Song, A. &amp; Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).`
│   │   inner_content: `Shirvani, A., Kalajian, T. A., Song, A. &amp; Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Gandolfo, L. C. &amp; Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).`
│   inner_content: `Gandolfo, L. C. &amp; Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Gandolfo, L. C. &amp; Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).`
│   │   inner_content: `Gandolfo, L. C. &amp; Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).`
│   inner_content: `Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).`
│   │   inner_content: `Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).`
│   inner_content: `Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).`
│   │   inner_content: `Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Lun, A. T. L., Chen, Y. &amp; Smyth, G. K. It&#39;s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. &amp; Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.`
│   inner_content: `Lun, A. T. L., Chen, Y. &amp; Smyth, G. K. It&#39;s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. &amp; Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Lun, A. T. L., Chen, Y. &amp; Smyth, G. K. It&#39;s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. &amp; Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.`
│   │   inner_content: `Lun, A. T. L., Chen, Y. &amp; Smyth, G. K. It&#39;s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. &amp; Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531`
│   inner_content: `Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531`
│   │   inner_content: `Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Langfelder, P. &amp; Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).`
│   inner_content: `Langfelder, P. &amp; Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Langfelder, P. &amp; Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).`
│   │   inner_content: `Langfelder, P. &amp; Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).`
│   inner_content: `Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).`
│   │   inner_content: `Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).`
│   inner_content: `Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).`
│   │   inner_content: `Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Kanehisa, M. &amp; Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).`
│   inner_content: `Kanehisa, M. &amp; Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Kanehisa, M. &amp; Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).`
│   │   inner_content: `Kanehisa, M. &amp; Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).`
│   inner_content: `Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).`
│   │   inner_content: `Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.`
│   inner_content: `Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.`
│   │   inner_content: `Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).`
│   inner_content: `Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).`
│   │   inner_content: `Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── NumListItemNode
│   full_match: `#Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).`
│   inner_content: `Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).`
│   depth: `1`
│   children:
│   ├── TextNode
│   │   full_match: `Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).`
│   │   inner_content: `Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).`├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}    Items in both categories MS and Genetics:`
│   inner_content: `{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}    Items in both categories MS and Genetics:`
│   level: `4`
│   children:
│   ├── SqlNode
│   │   full_match: `{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}`
│   │   inner_content: `SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;`
│   │   attrs_dict:
│   │   │   raw_content: db=&gt;vitamind
│   │   │   db: &gt;vitamind
│   ├── TextNode
│   │   full_match: `    Items in both categories MS and Genetics:`
│   │   inner_content: `    Items in both categories MS and Genetics:`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── CategoryNode
│   full_match: `{category id=&quot;18+72&quot; types=&quot;wiki&quot; sort=&quot;created_desc&quot; split=&quot;n&quot; and=&quot;y&quot; sub=&quot;n&quot; showdescription=&quot;n&quot; showname=&quot;y&quot; showtype=&quot;n&quot; one=&quot;y&quot; showTitle=&quot;n&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content: id=&quot;18+72&quot; types=&quot;wiki&quot; sort=&quot;created_desc&quot; split=&quot;n&quot; and=&quot;y&quot; sub=&quot;n&quot; showdescription=&quot;n&quot; showname=&quot;y&quot; showtype=&quot;n&quot; one=&quot;y&quot; showTitle=&quot;n&quot;
│   │   id: 18+72
│   │   types: wiki
│   │   sort: created_desc
│   │   split: n
│   │   and: y
│   │   sub: n
│   │   showdescription: n
│   │   showname: y
│   │   showtype: n
│   │   one: y
│   │   showTitle: n├── TextNode
│   full_match: `  `
│   inner_content: `  `├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!9 related items in VitaminDWiki`
│   inner_content: `9 related items in VitaminDWiki`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `9 related items in VitaminDWiki`
│   │   inner_content: `9 related items in VitaminDWiki`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022))`
│   inner_content: `((Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022))`
│   │   inner_content: `Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022`
│   │   page: `Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022`
│   │   │   inner_content: `Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `**Many virus have learned to protect themselves by deactivating the Vitamin D Receptor`
│   inner_content: `Many virus have learned to protect themselves by deactivating the Vitamin D Receptor`
│   depth: `2`
│   children:
│   ├── TextNode
│   │   full_match: `Many virus have learned to protect themselves by deactivating the Vitamin D Receptor`
│   │   inner_content: `Many virus have learned to protect themselves by deactivating the Vitamin D Receptor`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `**Which results in less Vitamin D getting to the cells`
│   inner_content: `Which results in less Vitamin D getting to the cells`
│   depth: `2`
│   children:
│   ├── TextNode
│   │   full_match: `Which results in less Vitamin D getting to the cells`
│   │   inner_content: `Which results in less Vitamin D getting to the cells`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Epigenetics of Multiple Sclerosis – March 2014))`
│   inner_content: `((Epigenetics of Multiple Sclerosis – March 2014))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Epigenetics of Multiple Sclerosis – March 2014))`
│   │   inner_content: `Epigenetics of Multiple Sclerosis – March 2014`
│   │   page: `Epigenetics of Multiple Sclerosis – March 2014`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Epigenetics of Multiple Sclerosis – March 2014`
│   │   │   inner_content: `Epigenetics of Multiple Sclerosis – March 2014`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies))`
│   inner_content: `((Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies))`
│   │   inner_content: `Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies`
│   │   page: `Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies`
│   │   │   inner_content: `Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))`
│   inner_content: `((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))`
│   │   inner_content: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`
│   │   page: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`
│   │   │   inner_content: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020))`
│   inner_content: `((UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020))`
│   │   inner_content: `UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020`
│   │   page: `UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020`
│   │   │   inner_content: `UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies))`
│   inner_content: `((Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies))`
│   │   inner_content: `Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies`
│   │   page: `Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies`
│   │   │   inner_content: `Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015))`
│   inner_content: `((7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015))`
│   │   inner_content: `7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015`
│   │   page: `7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015`
│   │   │   inner_content: `7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))`
│   inner_content: `((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))`
│   depth: `1`
│   children:
│   ├── LocalLinkNode
│   │   full_match: `((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))`
│   │   inner_content: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`
│   │   page: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`
│   │   │   inner_content: `Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021`├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!Several Health problems have a blunted response to Vitamin D supplementation`
│   inner_content: `Several Health problems have a blunted response to Vitamin D supplementation`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `Several Health problems have a blunted response to Vitamin D supplementation`
│   │   inner_content: `Several Health problems have a blunted response to Vitamin D supplementation`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*[https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342|Autism] 2018`
│   inner_content: `[https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342|Autism] 2018`
│   depth: `1`
│   children:
│   ├── LinkNode
│   │   full_match: `[https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342|Autism]`
│   │   inner_content: `Autism`
│   │   url: `https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Autism`
│   │   │   inner_content: `Autism`
│   ├── TextNode
│   │   full_match: ` 2018`
│   │   inner_content: ` 2018`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── ListItemNode
│   full_match: `*[https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491|Obesity]  Jan 2023`
│   inner_content: `[https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491|Obesity]  Jan 2023`
│   depth: `1`
│   children:
│   ├── LinkNode
│   │   full_match: `[https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491|Obesity]`
│   │   inner_content: `Obesity`
│   │   url: `https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Obesity`
│   │   │   inner_content: `Obesity`
│   ├── TextNode
│   │   full_match: `  Jan 2023`
│   │   inner_content: `  Jan 2023`├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}    Items in both categories MS and Vitamin D Receptor:`
│   inner_content: `{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}    Items in both categories MS and Vitamin D Receptor:`
│   level: `4`
│   children:
│   ├── SqlNode
│   │   full_match: `{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}`
│   │   inner_content: `SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;`
│   │   attrs_dict:
│   │   │   raw_content: db=&gt;vitamind
│   │   │   db: &gt;vitamind
│   ├── TextNode
│   │   full_match: `    Items in both categories MS and Vitamin D Receptor:`
│   │   inner_content: `    Items in both categories MS and Vitamin D Receptor:`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── CategoryNode
│   full_match: `{category id=&quot;18+142&quot; types=&quot;wiki&quot; sort=&quot;created_desc&quot; split=&quot;n&quot; and=&quot;y&quot; sub=&quot;n&quot; showdescription=&quot;n&quot; showname=&quot;y&quot; showtype=&quot;n&quot; one=&quot;y&quot; showTitle=&quot;n&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content: id=&quot;18+142&quot; types=&quot;wiki&quot; sort=&quot;created_desc&quot; split=&quot;n&quot; and=&quot;y&quot; sub=&quot;n&quot; showdescription=&quot;n&quot; showname=&quot;y&quot; showtype=&quot;n&quot; one=&quot;y&quot; showTitle=&quot;n&quot;
│   │   id: 18+142
│   │   types: wiki
│   │   sort: created_desc
│   │   split: n
│   │   and: y
│   │   sub: n
│   │   showdescription: n
│   │   showname: y
│   │   showtype: n
│   │   one: y
│   │   showTitle: n├── TextNode
│   full_match: `  `
│   inner_content: `  `├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!Strange that studies have not tried activating the VDR in people with MS`
│   inner_content: `Strange that studies have not tried activating the VDR in people with MS`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `Strange that studies have not tried activating the VDR in people with MS`
│   │   inner_content: `Strange that studies have not tried activating the VDR in people with MS`├── HorizontalRuleNode
│   full_match: `\n---\n`
│   inner_content: `---`├── HeadingNode
│   full_match: `!!!!VitaminDWiki - ((Vitamin D Receptor)) activation can be increased in many ways`
│   inner_content: `VitaminDWiki - ((Vitamin D Receptor)) activation can be increased in many ways`
│   level: `4`
│   children:
│   ├── TextNode
│   │   full_match: `VitaminDWiki - `
│   │   inner_content: `VitaminDWiki - `
│   ├── LocalLinkNode
│   │   full_match: `((Vitamin D Receptor))`
│   │   inner_content: `Vitamin D Receptor`
│   │   page: `Vitamin D Receptor`
│   │   children:
│   │   ├── TextNode
│   │   │   full_match: `Vitamin D Receptor`
│   │   │   inner_content: `Vitamin D Receptor`
│   ├── TextNode
│   │   full_match: ` activation can be increased in many ways`
│   │   inner_content: ` activation can be increased in many ways`├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── IncludeNode
│   full_match: `{include page=&quot;Vitamin D Receptor&quot; start=&quot;~tc~ startact ~/tc~&quot; stop=&quot;~tc~ endact ~/tc~&quot;}`
│   inner_content: ``
│   attrs_dict:
│   │   raw_content:  page=&quot;Vitamin D Receptor&quot; start=&quot;~tc~ startact ~/tc~&quot; stop=&quot;~tc~ endact ~/tc~&quot;
│   │   page: Vitamin D Receptor
│   │   start: ~tc~ startact ~/tc~
│   │   stop: ~tc~ endact ~/tc~
│   children:
│   ├── TextNode
│   │   full_match: ``
│   │   inner_content: ``├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── TikiCommentNode
│   full_match: `~tc~ (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) ~/tc~`
│   inner_content: ` (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) `
│   children:
│   ├── TextNode
│   │   full_match: ` (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) `
│   │   inner_content: ` (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) `├── TextNode
│   full_match: `\n`
│   inner_content: `\n`├── TikiCommentNode
│   full_match: `~tc~ (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) ~/tc~`
│   inner_content: ` (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) `
│   children:
│   ├── TextNode
│   │   full_match: ` (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) `
│   │   inner_content: ` (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) `

Original Tiki:
-------------
{DIV(class=&quot;lefth4&quot;)}{maketoc Title=&quot;&quot;}{DIV}
---
!!!!Transcriptomics identifies blunted immunomodulatory effects of vitamin D in people with multiple sclerosis
Scientific Reports Vol 14, Article number: 1436 (2024)  https://doi.org/10.1038/s41598-024-51779-0
Wei Z. Yeh, Rodney Lea, Jim Stankovich, Sandeep Sampangi, Louise Laverick, Anneke Van der Walt, Vilija Jokubaitis, Melissa Gresle &amp; Helmut Butzkueven 

Vitamin D deficiency is a risk factor for developing multiple sclerosis (MS). However, the immune effects of vitamin D in people with MS are not well understood. We analyzed transcriptomic datasets generated by RNA sequencing of immune cell subsets (CD4+, CD8+ T cells, B cells, monocytes) from 33 healthy controls and 33 untreated MS cases. We utilized a traditional bioinformatic pipeline and weighted gene co-expression network analysis (WGCNA) to determine genes and pathways correlated with endogenous vitamin D. In controls, CD4+ and CD8+ T cells had 1079 and 1188 genes, respectively, whose expressions were correlated with plasma 25-hydroxyvitamin D level (P &lt; 0.05). Functional enrichment analysis identified association with TNF-alpha and MAPK signaling. In CD4+ T cells of controls, vitamin D level was associated with expression levels of several genes proximal to multiple sclerosis risk loci (P = 0.01). Genes differentially associated with endogenous vitamin D by case–control status were enriched in TNF-alpha signaling via NF-κB. WGCNA suggested a blunted response to vitamin D in cases relative to controls. Collectively, our findings provide further evidence for the immune effects of vitamin D, and demonstrate a differential immune response to vitamin D in cases relative to controls, highlighting a possible mechanism contributing to MS pathophysiology.
{FONT(size=&quot;20&quot;)}__{ATTACH(inline=&quot;1&quot; id=&quot;20669&quot; icon=&quot;1&quot;)}Download the PDF from VitaminDWiki {ATTACH}__{FONT}
---
!!!!!82 References
#Yeh, W. Z. et al. Immunoregulatory effects and therapeutic potential of vitamin D in multiple sclerosis. Br. J. Pharmacol. 177, 4113-4133 (2020).
#Veldman, C. M., Cantorna, M. T. &amp; DeLuca, H. F. Expression of 1,25-dihydroxyvitamin D3 receptor in the immune system. Arch. Biochem. Biophys. 374, 334-338 (2000).
#Booth, D. R. et al. Cistromic and genetic evidence that the vitamin D receptor mediates susceptibility to latitude-dependent autoimmune diseases. Genes Immun. 17, 213-219 (2016).
#Baeke, F. et al. Human T lymphocytes are direct targets of 1,25-dihydroxyvitamin D3 in the immune system. J. Steroid Biochem. Mol. Biol. 121, 221-227 (2010).
#Liu, P. T., Stenger, S., Tang, D. H. &amp; Modlin, R. L. Cutting edge: Vitamin D-mediated human antimicrobial activity against Myco­bacterium tuberculosis is dependent on the induction of cathelicidin. J. Immunol. 179, 2060-2063 (2007).
#Penna, G. &amp; Adorini, L. 1a,25-Dihydroxyvitamin D3 inhibits differentiation, maturation, activation, and survival of dendritic cells leading to impaired alloreactive T cell activation. J. Immunol. 164, 2405-2411 (2000).
#Jeffery, L. E. et al. 1,25-dihydroxyvitamin D3 and interleukin-2 combine to inhibit T cell production of inflammatory cytokines and promote development of regulatory T cells expressing CTLA-4 and FoxP3. J. Immunol. 183, 5458-5467 (2009).
#Prietl, B. et al. High-dose cholecalciferol supplementation significantly increases peripheral CD4+ Tregs in healthy adults without negatively affecting the frequency of other immune cells. Eur. J. Nutr. 53, 751-759 (2014).
#Allen, A. C. et al. A pilot study of the immunological effects of high-dose vitamin D in healthy volunteers. Mult. Scler. 18, 1797-1800 (2012).
#Munger, K. L., Levin, L. I., Hollis, B. W., Howard, N. S. &amp; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 296, 2832-2838 (2006).
#da Costa, D. S. M. M. et al. Vitamin D modulates different IL-17-secreting T cell subsets in multiple sclerosis patients. J. Neuroim- munol. 299, 8-18 (2016).
#Kickler, K., Ni Choileain, S., Williams, A., Richards, A. &amp; Astier, A. L. Calcitriol modulates the CD46 pathway in T cells. PLoS One 7, e48486 (2012).
#Jagannath, V. A. et al. Vitamin D for the management of multiple sclerosis. Cochrane Database Syst. Rev. https://doi.org/10.1002/ 14651858.CD008422.pub3 (2018).
#Hupperts, R. et al. Randomized trial of daily high-dose vitamin D3 in patients with RRMS receiving subcutaneous interferon (3-1a. Neurology https://doi.org/10.1212/WNL.0000000000008445 (2019).
#Subramanian, A. et al. Gene set enrichment analysis: A knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl. Acad. Sci. 102, 15545-15550 (2005).
#International Multiple Sclerosis Genetics Consortium. Multiple sclerosis genomic map implicates peripheral immune cells and microglia in susceptibility. Science 365, eaav7188 (2019).
#Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, 17 (2005).
#Langfelder, P. &amp; Horvath, S. Eigengene networks for studying the relationships between co-expression modules. BMC Syst. Biol. 1, 54 (2007).
#Kriegler, M., Perez, C., DeFay, K., Albert, I. &amp; Lu, S. D. A novel form of TNF/cachectin is a cell surface cytotoxic transmembrane protein: Ramifications for the complex physiology of TNF. Cell 53, 45-53 (1988).
#Calzascia, T. et al. TNF-a is critical for antitumor but not antiviral T cell immunity in mice. J. Clin. Investig. 117, 3833-3845 (2007).
#Park, K. M. &amp; Bowers, W. J. Tumor necrosis factor-alpha mediated signaling in neuronal homeostasis and dysfunction. Cell Signal. 22, 977-983 (2010).
#Schioppa, T. et al. B regulatory cells and the tumor-promoting actions of TNF-a during squamous carcinogenesis. Proc. Natl. Acad. Sci. 108, 10662-10667 (2011).
#Sharief, M. K. &amp; Hentges, R. Association between tumor necrosis factor-a and disease progression in patients with multiple scle­rosis. N. Engl. J. Med. 325, 467-472 (1991).
#Rieckmann, P. et al. Tumor necrosis factor-a messenger RNA expression in patients with relapsing-remitting multiple sclerosis is associated with disease activity. Ann. Neurol. 37, 82-88 (1995).
#Huang, W.-X., Huang, P., Link, H. &amp; Hillert, J. Cytokine analysis in multiple sclerosis by competitive RT-PCR: A decreased expres­sion of IL-10 and an increased expression of TNF-a in chronic progression. Mult. Scler. 5, 342-348 (1999).
#van Oosten, B. W. et al. Increased MRI activity and immune activation in two multiple sclerosis patients treated with the mono­clonal anti-tumor necrosis factor antibody cA2. Neurology 47, 1531-1534 (1996).
#Arnett, H. A. et al. TNFa promotes proliferation of oligodendrocyte progenitors and remyelination. Nat. Neurosci. 4, 1116-1122 (2001).
#Müller, K. et al. 1,25-dihydroxyvitamin D3 inhibits cytokine production by human blood monocytes at the post-transcriptional level. Cytokine 4, 506-512 (1992).
#Lysandropoulos, A. P. et al. Vitamin D has a direct immunomodulatory effect on CD8+ T cells of patients with early multiple sclerosis and healthy control subjects. J. Neuroimmunol. 233, 240-244 (2011).
#Peterson, C. A. &amp; Heffernan, M. E. Serum tumor necrosis factor-alpha concentrations are negatively correlated with serum 25(OH) D concentrations in healthy women. J. Inflamm. 5, 10 (2008).
#Yang, W. S. et al. 1,25-Dihydroxyvitamin D3 causes ADAM10-dependent ectodomain shedding of tumor necrosis factor receptor 1 in vascular smooth muscle cells. Mol. Pharmacol. 87, 533-542 (2015).
#Song, G. G., Bae, S.-C. &amp; Lee, Y. H. Association between vitamin D intake and the risk of rheumatoid arthritis: A meta-analysis. Clin. Rheumatol. 31, 1733-1739 (2012).
#Ananthakrishnan, A. N. et al. Higher predicted vitamin d status is associated with reduced risk of Crohn&#39;s disease. Gastroenterology 142, 482-489 (2012).
#Kamen, D. L. et al. Vitamin D deficiency in systemic lupus erythematosus. Autoimmun. Rev. 5, 114-117 (2006).
#Umar, M. et al. Vitamin D and the pathophysiology of inflammatory skin diseases. SPP 31, 74-86 (2018).
#Hahn, J. et al. Vitamin D and marine omega 3 fatty acid supplementation and incident autoimmune disease: VITAL randomized controlled trial. BMJ 376, e066452 (2022).
#Dimitrov, V. et al. Vitamin D-regulated gene expression profiles: Species-specificity and cell-specific effects on metabolism and immunity. Endocrinology 162, bqaa218 (2021).
#von Essen, M. R. et al. Vitamin D controls T cell antigen receptor signaling and activation of human T cells. Nat. Immunol. 11, 344-349 (2010).
#Richards, J. B. et al. Higher serum vitamin D concentrations are associated with longer leukocyte telomere length in women. Am. J. Clin. Nutr. 86, 1420-1425 (2007).
#Jirmanova, L., Giardino Torchia, M. L., Sarma, N. D., Mittelstadt, P. R. &amp; Ashwell, J. D. Lack of the T cell-specific alternative p38 activation pathway reduces autoimmunity and inflammation. Blood 118, 3280-3289 (2011).
#Zhang, Y. et al. Vitamin D inhibits monocyte/macrophage proinflammatory cytokine production by targeting MAPK phos­phatase-1. J. Immunol. 188, 2127-2135 (2012).
#Ding, C., Wilding, J. P H. &amp; Bing, C. 1,25-Dihydroxyvitamin D3 protects against macrophage-induced activation of NFkB and MAPK signalling and chemokine release in human adipocytes. PLoS One 8, e61707 (2013).
#Xin, L. et al. 1,25-Dihydroxy vitamin D3 attenuates the oxidative stress-mediated inflammation induced by PM2.5 via the p38/ NF-kB/NLRP3 pathway. Inflammation 42, 702-713 (2019).
#Joshi, S. et al. 1,25-Dihydroxyvitamin D3 ameliorates Th17 autoimmunity via transcriptional modulation of interleukin-17A. Mol. Cell. Bid. 31, 3653-3669 (2011).
#Takeuchi, A. et al. Nuclear factor of activated T cells (NFAT) as a molecular target for 1a,25-dihydroxyvitamin D3-mediated effects. J. Immunol. 160, 209-218 (1998).
#Matilainen, J. M., Räsänen, A., Gynther, P. &amp; Väisänen, S. The genes encoding cytokines IL-2, IL-10 and IL-12B are primary 1a,25(OH)2D3 target genes. J. Steroid Biochem. Mol. Biol. 121, 142-145 (2010).
#Alroy, I., Towers, T. L. &amp; Freedman, L. P. Transcriptional repression of the interleukin-2 gene by vitamin D3: Direct inhibition of NFATp/AP-1 complex formation by a nuclear hormone receptor. Mol. Cell. Biol. 15, 5789-5799 (1995).
#Ikeda, U. et al. 1a,25-Dihydroxyvitamin D3 and all-trans retinoic acid synergistically inhibit the differentiation and expansion of Th17 cells. Immunol. Lett. 134, 7-16 (2010).
#Palmer, M. T. et al. Lineage-specific effects of 1,25-dihydroxyvitamin d3 on the development of effector CD4 T cells. J. Biol. Chem. 286, 997-1004 (2011).
#Müller, K., 0dum, N. &amp; Bendtzen, K. 1,25-Dihydroxyvitamin D3 selectively reduces interleukin-2 levels and proliferation of human T cell lines in vitro. Immunol. Lett. 35, 177-182 (1993).
#Urry, Z. et al. The role of 1a,25-dihydroxyvitamin D3 and cytokines in the promotion of distinct Foxp3+ and IL-10+ CD4+ T cells. Eur. J. Immunol. 42, 2697-2708 (2012).
#Chauss, D. et al. Autocrine vitamin D signaling switches off pro-inflammatory programs of TH1 cells. Nat. Immunol. 23, 62-74 (2022).
#Bhargava, P. et al. Multiple sclerosis patients have a diminished serologic response to vitamin D supplementation compared to healthy controls. Mult. Scler. 22, 753-760 (2016).
#Bhargava, P., Fitzgerald, K. C., Calabresi, P. A. &amp; Mowry, E. M. Metabolic alterations in multiple sclerosis and the impact of vitamin D supplementation. JCI Insight 2, e95302 (2017).
#Lu, M., Shi, H., Taylor, B. V. &amp; Körner, H. Alterations of subset and cytokine profile of peripheral T helper cells in PBMCs from Multiple Sclerosis patients or from individuals with MS risk SNPs near genes CYP27B1 and CYP24A1. Cytokine 153, 155866 (2022).
#Mu, Z. et al. The impact of cell type and context-dependent regulatory variants on human immune traits. Genome Biol. 22, 122 (2021).
#Ewing, E. et al. Combining evidence from four immune cell types identifies DNA methylation patterns that implicate functionally distinct pathways during Multiple Sclerosis progression. EBioMedicine 43, 411-423 (2019).
#Kim, D. et al. Peripheral T-cells, B-cells, and monocytes from multiple sclerosis patients supplemented with high-dose vitamin d show distinct changes in gene expression profiles. Nutrients 14, 4737 (2022).
#Polman, C. H. et al. Diagnostic criteria for multiple sclerosis: 2010 Revisions to the McDonald criteria. Ann. Neurol. 69, 292-302 (2011).
#Andrews, S. FastQC: A Quality Control Tool for High Throughput Sequence Data (2010).
#Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525-527 (2016).
#R Core Team. R: A Language and Environment for Statistical Computing (2019).
#Soneson, C., Love, M. I. &amp; Robinson, M. D. Differential analyses for RNA-seq: Transcript-level estimates improve gene-level infer­ences. F1000Res 4, 1521 (2016).
#Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: A Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140 (2010).
#Risso, D., Ngai, J., Speed, T. P. &amp; Dudoit, S. Normalization of RNA-seq data using factor analysis of control genes or samples. Nat. Biotechnol. 32, 896-902 (2014).
#Eisenberg, E. &amp; Levanon, E. Y. Human housekeeping genes, revisited. Trends Genet. 29, 569-574 (2013).
#Ramagopalan, S. V. et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 20, 1352-1360 (2010).
#Neme, A., Seuter, S. &amp; Carlberg, C. Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta (BBA) Gene Regul. Mech. 1860, 952-961 (2017).
#Shirvani, A., Kalajian, T. A., Song, A. &amp; Holick, M. F. Disassociation of vitamin Ds calcemic activity and non-calcemic genomic activity and individual responsiveness: A randomized controlled double-blind clinical trial. Sci. Rep. 9, 17685 (2019).
#Gandolfo, L. C. &amp; Speed, T. P RLE plots: Visualizing unwanted variation in high dimensional data. PLoS One 13, e0191629 (2018).
#Cunningham, F. et al. Ensembl 2022. Nucleic Acids Res. 50, D988-D995 (2022).
#Durinck, S., Spellman, P. T., Birney, E. &amp; Huber, W. Mapping identifiers for the integration of genomic datasets with the R/Bio- conductor package biomaRt. Nat. Protoc. 4, 1184-1191 (2009).
#Lun, A. T. L., Chen, Y. &amp; Smyth, G. K. It&#39;s DE-licious: A recipe for differential expression analyses of RNA-seq experiments using quasi-likelihood methods in edgeR. In Statistical Genomics: Methods and Protocols (eds Mathe, E. &amp; Davis, S.) 391-416 (Springer, 2016). https://doi.org/10.1007/978-1-4939-3578-9_19.
#Evelo, C. et al. Vitamin D metabolism (Homo sapiens)—WikiPathways. https://www.wikipathways.org/index.php/PathwayWP1531
#Langfelder, P. &amp; Horvath, S. WGCNA: An R package for weighted correlation network analysis. BMC Bioinform. 9, 559 (2008).
#Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: A practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B (Methodology) 57, 289-300 (1995).
#Liberzon, A. et al. Molecular signatures database (MSigDB) 3.0. Bioinformatics 27, 1739-1740 (2011).
#Kanehisa, M. &amp; Goto, S. KEGG: Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 28, 27-30 (2000).
#Wu, T. et al. clusterProfiler 4.0: A universal enrichment tool for interpreting omics data. Innovation 2, 100141 (2021).
#Wickham, H. ggplot2 (Springer International Publishing, 2016). https://doi.org/10.1007/978-3-319-24277-4.
#Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritiza­tion. Nucleic Acids Res. 37, W305-W311 (2009).
#Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinfor­matics 32, 2847-2849 (2016).
---
!!!!{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 72 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}    Items in both categories MS and Genetics:
{category id=&quot;18+72&quot; types=&quot;wiki&quot; sort=&quot;created_desc&quot; split=&quot;n&quot; and=&quot;y&quot; sub=&quot;n&quot; showdescription=&quot;n&quot; showname=&quot;y&quot; showtype=&quot;n&quot; one=&quot;y&quot; showTitle=&quot;n&quot;}  
---
!!!!9 related items in VitaminDWiki
*((Epstein-Barr virus increases risk of Multiple Sclerosis by 32X - Jan 2022))
**Many virus have learned to protect themselves by deactivating the Vitamin D Receptor
**Which results in less Vitamin D getting to the cells
*((Epigenetics of Multiple Sclerosis – March 2014))
*((Resveratrol treats Multiple Sclerosis and other autoimmune diseases – many studies))
*((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))
*((UK people with Multiple Sclerosis are 3X more likely to take Vitamin D - Oct 2020))
*((Multiple Sclerosis much more likely if poor Vitamin D Receptor – several studies))
*((7 ng less response by Multiple Sclerosis patients to 5,000 IU of vitamin D – Aug 2015))
*((Gene variants can reduce Vitamin D response by 1.7X (14,000 IU daily, Multiple Sclerosis) – Dec 2021))
---
!!!!Several Health problems have a blunted response to Vitamin D supplementation
*[https://www.tandfonline.com/doi/abs/10.1080/1028415X.2018.1529342|Autism] 2018
*[https://jamanetwork.com/journals/jamanetworkopen/article-abstract/2800491|Obesity]  Jan 2023
---
!!!!{SQL(db=&gt;vitamind)}SELECT COUNT(*) FROM (SELECT catObjectId, COUNT(*) FROM `tiki_category_objects` WHERE categId = 18  OR categId = 142 GROUP BY catObjectId HAVING COUNT(*) &gt; 1) AS categs INNER JOIN tiki_objects ON tiki_objects.objectId = categs.catObjectId WHERE type = &quot;wiki page&quot;{SQL}    Items in both categories MS and Vitamin D Receptor:
{category id=&quot;18+142&quot; types=&quot;wiki&quot; sort=&quot;created_desc&quot; split=&quot;n&quot; and=&quot;y&quot; sub=&quot;n&quot; showdescription=&quot;n&quot; showname=&quot;y&quot; showtype=&quot;n&quot; one=&quot;y&quot; showTitle=&quot;n&quot;}  
---
!!!!Strange that studies have not tried activating the VDR in people with MS
---
!!!!VitaminDWiki - ((Vitamin D Receptor)) activation can be increased in many ways
{include page=&quot;Vitamin D Receptor&quot; start=&quot;~tc~ startact ~/tc~&quot; stop=&quot;~tc~ endact ~/tc~&quot;}
~tc~ (alias(People with Multiple Sclerosis have poor response to Vitamin D supplementation - Jan 2024)) ~/tc~
~tc~ (alias(People with Multiple Sclerosis have blunted response to Vitamin D supplementation - Jan 2024)) ~/tc~
</code>
</pre>
