{
    "title": "Reasons why the virus might mutate and become immune to the COVID-19 vaccine",
    "slug": "reasons-why-the-virus-might-mutate-and-become-immune-to-the-covid-19-vaccine",
    "aliases": [
        "/Reasons+why+the+virus+might+mutate+and+become+immune+to+the+COVID-19+vaccine+\u2013+Nov+2020",
        "/12267"
    ],
    "tiki_page_id": 12267,
    "date": "2021-01-27",
    "categories": [
        "Virus"
    ],
    "tags": [
        "Virus",
        "antibiotics",
        "bacteria",
        "immunity",
        "influenza",
        "pfizer",
        "tuberculosis",
        "vaccination",
        "virus"
    ]
}


#### Monitor for COVID-19 vaccine resistance evolution during clinical trials

PLoS Biol 18(11):e3001000. https:// doi.org/10.1371/journal.pbio.3001000

David A. KennedylD1  dak30@psu.edu, Andrew F. ReadnD1,2

1 Centerfor Infectious Disease Dynamics, Departmentof Biology, The PennsylvaniaState University, Pennsylvania, United States ofAmerica,

2 Departmentof Entomology, The PennsylvaniaState University, Pennsylvania, United States ofAmerica

<div class="border" style="background-color:#FFFAE2;padding:15px;margin:10px 0;border-radius:5px;width:800px">

It appears that viral experts are aware that virus can evolve to become immune from a vaccine.

---

* [Worrisome New Coronavirus Strains Are Emerging. Why Now?](https://www.wired.com/story/worrisome-new-coronavirus-strains-are-emerging-why-now/) Wired Jan 27

   * three distinct versions of the virus were found in UK, SA, and Brazil

   * "That pattern is what scientists refer to as “convergent evolution,” and it’s a sign of trouble ahead."

   * "These different lineages are essentially arriving at the same solution for how to interact more efficiently with the human receptor, ACE2.”

   * . . "estimate that the UK variant, alternatively known as B.1.1.7, is between 30 and 50 percent more infectious than other circulating strains."

   * Manaus Brazil had achived herd immunity last Sping to previous versions, but are becoming infected again by a new mutation

   * ". . .antibodies taken from people vaccinated with either Pfizer/BioNTech or Moderna’s shots were  **up to three times less effective**  at neutralizing the pseudoviruses carrying the mutations found in B.1.351. . ."

* [Vaccines Are Pushing Pathogens to Evolve – May 2018](/posts/vaccines-are-pushing-pathogens-to-evolve)

</div>

 **<i class="fas fa-file-pdf" style="margin-right: 0.3em;"></i><a href="https://d378j1rmrlek7x.cloudfront.net/attachments/pdf/monitor-for-covid-19-vaccine-resistance-evolution-during-clinical-trials---same-author.pdf">Download the PDF from VitaminDWiki</a>** 

Although less common than the evolution of antimicrobial drug resistance, vaccine resistance can and has evolved. How likely is it that COVID-19vaccines currently in development will be undermined by viral evolution? We argue that this can be determined by repurposing samples that are already being collected as part of clinical trials. Such information would be useful for prioritizing investment among candidate vaccines and maximizing the potential long-term impact of COVID-19 vaccines.

A safe and effective vaccine against COVID-t9 would go a long way towards helping society return to its pre-pandemic normal. According to the World Health Organization, at least C98 COVID-19 vaccines are currently in the development pipeline, with 44 currently undergoing clinical evaluation ⑴. That evaluation is, rightly so, focused on safety and efficacy. Here, we advocate for moderate additional effort during clinical trials to collect and publish data that can inform the risk of resistance evolution.

Much like antimicrobial drug resistance, vaccine resistance can and does evolve <sup>[2]</sup>. When it does evolve, vaccine resistance is achieved through mechanisms such as serotype replacement <sup>[3]</sup>, antigenic change <sup>[4]</sup>, or increases in disease severity <sup>[5]</sup>. However, for many vaccines, the evolution of resistance has never occurred <sup>[6]</sup>. For example, the measles vaccine has been widely used for decades without the virus ever-evolving the ability to transmit through vaccinated hosts. Similarly, smallpox was completely eradicated, in large part due to vaccination that viral evolution failed to overcome. In contrast, Streptococcuspneumoniae quickly evolved resistance to the pneumococcal conjugate vaccine (PCV7), necessitating the development and deployment of a new vaccine, PCV13 <sup>[7]</sup>. Recently, the features that are critical to delaying the evolution of vaccine resistance have been described <sup>[6]</sup>. Here, we argue that by repurposing standard samples from COVID-19 clinical trials, the potential for vaccine resistance can be assessed even before vaccine licensure.

To our knowledge,  **all documented cases of vaccine resistance can be attributed to the absence of at least one of three key features**  that most vaccines possess: 

* 1) the vaccine induces an immune response that protects hosts by targeting multiple virus epitopes simultaneously, thereby generating redundant and evolutionarily-robust protection, 

   * When feature 1 is present, resistance would likely require the appearance of multiple mutations, as opposed to just one, on the same genetic background. 

* 2) the vaccine suppresses pathogen growth within hosts and stops transmission from vaccine-protected hosts, and 

   * When feature 2 is present, little pathogen diversity would be generated during pathogen growth within vaccinated hosts, and the effects of selection on any resistance mutations that arose would be minimal.

* 3) the vaccine-induced immune response protects against all circulating serotypes of the target pathogen. 

   * When feature 3 is present, new virus variants would need to be generated before resistance could be a problem, since vaccine resistance does not pre-exist. 

Combined together, these three features make the probability of resistance emergence vanishingly small <sup>[6]</sup>.

It is important that the probability of resistance evolution be small because vaccine resistance can negatively impact public health. While antimicrobial drugs can be tailored to individual patients at the time of treatment, the choice of which vaccine to administer must be made well in advance of pathogen exposure. Should vaccine resistance emerge in the weeks, months, or years between vaccination and exposure, a vaccinated individual could be left unprotected. Should resistance become widespread and common, entire vaccination campaigns could retroactively be rendered ineffective. Moreover, since pre-existing antibodies frequently interfere with vaccine efficacy <sup>[8]</sup>, we cannot assume that a new vaccine would be capable of restoring protection. Additionally, a large fraction of COVID-19 candidate vaccines target the spike protein of the virus or the receptor binding domain of the spike protein <sup>[9]</sup>, and so the evolution of vaccine resistance against one vaccine could simultaneously undermine others, an outcome referred to as ‘collateral1 or ‘cross1 resistance in the case of antimicrobial drugs.

To avoid being caught off guard by the evolution of vaccine resistance, standard samples from clinical trials can be repurposed to assess the risk of resistance evolution even before a vaccine is licensed (Fig 1). First, blood samples are collected during almost all COVID-19 clinical trials to quantify individual responses to vaccination through antibody titer and serum neutralization tests. We propose that in addition to performing these tests, blood samples also be used to quantify the redundancy of immune protection generated by candidate vaccines <sup>[10,11]</sup>. Since redundant immune protection delays the evolution of vaccine resistance, much the same as combination drug therapy delays the evolution of antibiotic resistance, it is critical that vaccination induces immune responses against multiple non-overlapping viral epitopes. For SARS-CoV-2, as in other systems, resistance has already been shown to evolve quickly against monoclonal neutralizing antibodies relative to combinations of these antibodies <sup>[12]</sup>. Although yet to be shown for SARS-CoV-2, diverse T-cell responses can similarly delay resistance evolution <sup>[7]</sup>. Therefore, quantifying the redundancy of immune protection generated by vaccination is key information for determining the likelihood of resistance evolution.

Second, many COVID-19 vaccine clinical trials collect weekly nasal swabs or fecal samples from vaccinated and control individuals to quantify vaccine protection against infection. We propose that these samples also be used to collect viral titer data as indicators of transmission potential. Strongly suppressing pathogen transmission through vaccinated hosts is key to preventing the spread of partial resistance should it arise, since it reduces the opportunities for selection to act <sup>[6]</sup>. While viral titer data are imperfect measures of transmission, they are a readily collectible proxy. Note that extra effort to collect higher quality transmission data may also be justifiable given the value of transmission data for optimizing vaccine distribution <sup>[13]</sup>.

Third, many COVID-19 clinical trials collect nasopharyngeal swab samples from symptomatic vaccinated and control individuals to confirm SARS-CoV-2 as the causative agent of illness. We propose that viral genome sequences be generated from these swabs to look for evidence of vaccine-driven selection.

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/monitor-f1.jpg" alt="image" width="300">

 **Fig 1.**   Schematic illustrating three ways that standard samples from COVID-19 clinical trials can be repurposed to assess the risk that vaccine resistance will evolve. 

* 1. The complexity ofB-cell and T-cell responses can be measured using blood samples <sup>[10,11]</sup>. Different neutralizing antibodies are depicted above in different colors. More complex responses indicate more evolutionarily robust immunity. 

* 2. The effect of vaccination on transmission potential can be assessed by collecting viral titer data using routine nasal swabs. Plaque assays from multiple vaccinated and control individuals are compiled into a histogram. Undetectable viral titers suggest little or no transmission potential, due to either complete immune protection or the absence of exposure. High viral titers suggest high transmission potential due to the absence of a protective immune response. Intermediate viral titers, marked above with an asterisk, suggest moderate transmission potential due to partial vaccine protection. Intermediate titers indicate an increased risk for resistance evolution since pathogen diversity can be generated within hosts and selection can act during transmission between hosts.

* 3. Pre-existing variation for vaccine resistance can be assessed by recovering genome sequences from nasopharyngeal swabs of symptomatic COVID-19 cases included in the study. In a placebo-controlled, double-blind study, any significant differences in the genome sequences of samples from vaccinated and control individuals would suggest at least partial vaccine resistance.

For example, differences in allele frequencies between the viral genomes collected from vaccinated and control individuals would indicate selection <sup>[14]</sup>, while simultaneously alleging a genetic basis for resistance <sup>[2, 3]</sup>. If such evidence were seen during a clinical study, as it can be <sup>[3]</sup>, it would strongly indicate the potential for resistance to evolve.

A safe and effective COVID-19 vaccine is a priority, and it is urgently needed. Given this, we are not advocating to delay the release of a COVID-19 vaccine that is safe and efficacious even if there is a high likelihood that resistance will evolve against it. Rather, we are advocating that all vaccines be assessed as early as possible for the likelihood they will drive resistance evolution. As we explain above, this assessment can be conducted in a controlled manner during clinical trials, rather than first waiting for promising trial results to melt away after a vaccine is licensed.

For other diseases, vaccine failure due to pathogen evolution has occurred both during clinical trials <sup>[14]</sup> and after licensure <sup>[2]</sup>. We therefore suggest that the risk of resistance be used to prioritize investment among otherwise similarly promising vaccine candidates. If all first-generation vaccines are at appreciable risk of being undermined by virus evolution, it will be important to continue additional COVID-19 vaccine development following the discovery of a first, safe and efficacious vaccine. Predicting when and how resistance will be likely to evolve will give important insight into what needs to be monitored in phase IV studies after vaccine roll-out <sup>[3,7]</sup>.

The world needs a COVID-19 vaccine urgently, just as the world previously needed drugs against tuberculosis and human immunodeficiency virus (HIV). It is tempting to leave evolutionary concerns until after a vaccine is introduced. But as we saw in the case of tuberculosis and HIV, the evolution of resistance can quickly undermine newly discovered interventions. By learning from solutions to previous evolutionary challenges, we can do better for COVID- 19.

#### References

1. World Health Organization. Draft landscape of COVID-19 candidate vaccines. 19 October 2020. <span>[cited 2020 Oct 23]</span>. Available from: https://www.who.int/publications/m/item/draft-landscape-of-covid-19- candidate-vaccines.

1. Kennedy DA, Read AF. Why the evolution of vaccine resistance is less of a concern than the evolution of drug resistance. Proc NatlAcad Sci USA. 2018; 115(51):12878-12886. https://doi.org/10.1073/ pnas.1717159115 PMID: 30559199

   * <i class="fas fa-file-pdf" style="margin-right: 0.3em;"></i><a href="https://d378j1rmrlek7x.cloudfront.net/attachments/pdf/monitor-for-covid-19-vaccine-resistance-evolution-during-clinical-trials---same-author.pdf">Download the PDF from VitaminDWiki</a>

1. Lipsitch M. Vaccination against colonizing bacteria with multiple serotypes. Proc Natl Acad Sci USA. 1997; 94(12):6571-6576. https://doi.org/10.1073/pnas.94.12.6571 PMID: 9177259

1. OctaviaS, Maharjan RP, SintchenkoV, Stevenson G, Reeves PR, GilbertGL, etal. Insight into evolution of Bot detella pertussis from comparative genomic analysis: evidence of vaccine-driven selection. MolBiol Evol.2011; 28(1):707-715. https://doi.org/10.1093/molbev/msq245 PMID: 20833694

1. Read AF, Baigent SJ, Powers C, Kgosana LB, Blackwell L, Smith LP, et al. Imperfect vaccination can enhance the transmission of highlyvirulent pathogens. PLoS Biol. 2015; 13(7):e1002198. https://doi.org/10.1371/journal.pbio.1002198 PMID: 26214839

1. Kennedy DA, ReadAF. Why does drug resistance readily evolve but vaccine resistance does not? Proc RoyalSocB.2017; 284:20162562. https://doi.org/10.1098/rspb.2016.2562 PMID: 28356449

1. Weinberger DM, Malley R, Lipsitch M. Serotype replacement in disease after pneumococcal vaccination. Lancet.2011; 378(10):1962-1973. https://doi.org/10.1016/S0140-6736(10)62225-8 PMID: 21492929

1. Dagan R, Poolman J, SiegristC-A. Glycoconjugate vaccines and immune interference: a review. Vac- cine.2010; 28:5513-5523. https://doi.org/10.1016/j.vaccine.2010.06.026 PMID: 20600514

1. Krammer F. SARS-CoV-2 vaccines in development. Nature. 2020; 586:516-527. https://doi.org/10.1038/s41586-020-2798-3 PMID: 32967006

1. Lehmann PV, Suwansaard M, ZhangT, Roen DR, Kirchenbaum GA, Karulin AY, etal. Comprehensive evaluation of the expressed CD8+ T cell epitope space using high-throughput epitope mapping. Front Immunol.2019; 10:655. https://doi.org/10.3389/fimmu.2019.00655 PMID: 31105686

1. Rogers TF, Zhao F, Huang D, Beutler N, Burns A, He W-t, et al. 2020. Isolation of potent SARS-CoV-2 neutralizing antibodies and protection from disease in a small animal model. Science. 2020; 369 (6506):956-963. https://doi.org/10.1126/science.abc7520 PMID: 32540903

1. Baum A, Fulton BO, Wloga E, Copin R, Pascal KE, Russo V, et al. Antibody cocktail to SARS-CoV-2 spike protein prevents rapid mutational escape seen with individual antibodies. Science. 2020; 369 (6506):1014-1018.

1. MedlockJ, GalvaniAP. Optimizing influenza vaccine distribution. Science. 2009; 325(5948):1705- 1708. https://doi.org/10.1126/science.1175570 PMID: 19696313

1. Genton B, Betuela I, Felger I, Al-Yaman F, Anders RF, Saul A, et al. A recombinant blood-stage malaria vaccine reduces Plasmodium falciparum density and exerts selective pressure on parasite populations in a phase 1-2btrial in Papua NewGuinea. J Infect Dis. 2002; 185(6):820-827. https://doi.org/10.1086/339342 PMID: 11920300

<!-- ~tc~ (alias(Reasons why the virus might mutate and become immume to the COVID-19 vaccine – Nov 2020)) ~/tc~ -->