{
    "title": "Trying to understand the effect of genetics vitamin D levels in the blood",
    "slug": "trying-to-understand-the-effect-of-genetics-vitamin-d-levels-in-the-blood",
    "aliases": [
        "/Trying+to+understand+the+effect+of+genetics+vitamin+D+levels+in+the+blood+\u2013+Jan+2013",
        "/4167"
    ],
    "tiki_page_id": 4167,
    "date": "2013-06-01",
    "categories": [
        "Genetics"
    ],
    "tags": [
        "Genetics",
        "vitamin d",
        "vitamin d blood test"
    ]
}


## Exploring the genetic architecture of circulating 25-hydroxyvitamin D.

Genet Epidemiol. 2013 Jan;37(1):92-8. [doi: 10.1002/gepi.21694.](https://doi.org/10.1002/gepi.21694.) Epub 2012 Nov 7.

Hiraki LT, Major JM, Chen C, Cornelis MC, Hunter DJ, Rimm EB, Simon KC, Weinstein SJ, Purdue MP, Yu K, Albanes D, Kraft P.

Program in Molecular and Genetic Epidemiology, Harvard School of Public Health, Boston, Massachusetts 02215, USA. lindahiraki@mail.harvard.edu

The primary circulating form of vitamin D is 25-hydroxy vitamin D (25(OH)D), a modifiable trait linked with a growing number of chronic diseases. In addition to environmental determinants of 25(OH)D, including dietary sources and skin ultraviolet B (UVB) exposure, twin- and family-based studies suggest that genetics contribute substantially to vitamin D variability with heritability estimates ranging from 43% to 80%. 

Genome-wide association studies (GWAS) have identified single nucleotide polymorphisms (SNPs) located in four gene regions associated with 25(OH)D. These SNPs collectively explain only a fraction of the heritability in 25(OH)D estimated by twin- and family-based studies. Using 25(OH)D concentrations and GWAS data on 5,575 subjects drawn from five cohorts, we hypothesized that genome-wide data, in the form of (1) a polygenic score comprised of hundreds or thousands of SNPs that do not individually reach GWAS significance, or (2) a linear mixed model for genome-wide complex trait analysis, would explain variance in measured circulating 25(OH)D beyond that explained by known genome-wide significant 25(OH)D-associated SNPs. GWAS identified SNPs explained 5.2% of the variation in circulating 25(OH)D in these samples and there was little evidence additional markers significantly improved predictive ability. On average, a polygenic score comprised of GWAS-identified SNPs explained a larger proportion of variation in circulating 25(OH)D than scores comprised of thousands of SNPs that were on average, nonsignificant. Employing a linear mixed model for genome-wide complex trait analysis explained little additional variability (range 0-22%). The absence of a significant polygenic effect in this relatively large sample suggests an oligogenetic architecture for 25(OH)D.

Â© 2012 WILEY PERIODICALS, INC.

PMID:     23135809

 **<span style="color:#00F;">PDF is attached at the bottom of this page</span>**