{
    "title": "Vitamin D Nutrigenomics - High, Medium, and Low Responders",
    "slug": "vitamin-d-nutrigenomics-high-medium-and-low-responders",
    "aliases": [
        "/Vitamin+D+Nutrigenomics+-+High+Medium+and+Low+Responders+-+March+2019",
        "/10601"
    ],
    "tiki_page_id": 10601,
    "date": "2019-03-22",
    "categories": [
        "Genetics",
        "Predict",
        "Vit D Receptor",
        "Vit D Binding Protein"
    ],
    "tags": [
        "Genetics",
        "IBD",
        "Predict",
        "Vit D Binding Protein",
        "Vit D Receptor",
        "asthma",
        "asthma receptor",
        "autoimmune",
        "autoimmune and pregnancy",
        "bacteria",
        "bone",
        "bone cancer",
        "breathing",
        "cancer",
        "cancer in VDR",
        "child",
        "childhood cancer",
        "cholesterol",
        "colon cancer",
        "dosage",
        "genetics",
        "gut",
        "health risk",
        "high dose",
        "immunity",
        "infection",
        "intervention",
        "kidney",
        "multiple sclerosis",
        "multiple sclerosis and pregnancy",
        "osteoporosis",
        "pediatric autoimmune",
        "pregnancy",
        "rickets",
        "skin cancer",
        "therapeutic intervention",
        "tuberculosis",
        "urinary tract infection",
        "vitamin d",
        "vitamin d blood test",
        "vitamin d receptor",
        "vitamin k"
    ]
}


**[128 citations of this study in Google Scholar as of Dec 2022](https://scholar.google.com/scholar?cites=2592429304506722280&as_sdt=5,48&sciodt=0,48&hl=en)** 

* Vitamin D in the Context of Evolution - Carlberg - July 2022,  https://doi.org/10.3390/nu14153018 FREE PDF

   * 1.2 billion years of Vitamin D

</div>

 **<i class="fas fa-file-pdf" style="margin-right: 0.3em;"></i><a href="https://d378j1rmrlek7x.cloudfront.net/attachments/pdf/nutrigenomics-compressed.pdf">Download the PDF from VitaminDWiki</a>** 

 **Figure 2** 

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/n-fig-2.jpg" alt="image" width="700">

 **Figure 3** 

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/n-fig-3.jpg" alt="image" width="500">

 **Figure 4** 

<img src="https://d378j1rmrlek7x.cloudfront.net/attachments/jpeg/n-fig-4.jpg" alt="image" width="600">

Nutrigenomics studies how environmental factors, such as food intake and lifestyle, influence the expression of the genome. Vitamin D3 represents a master example of nutrigenomics, since via its metabolite 1a,25-dihydroxyvitamin D3, which binds with high-affinity to the vitamin D receptor, the secosteroid directly affects the epigenome and transcriptome at thousands of loci within the human genome. Vitamin D is important for both cellular metabolism and immunity, as it controls calcium homeostasis and modulates the response of the innate and adaptive immune system. At sufficient UV-B exposure, humans can synthesize vitamin D3 endogenously in their skin, but today's lifestyle often makes the molecule a true vitamin and micronutrient that needs to be taken up by diet or supplementation with pills. The individual's molecular response to vitamin D requires personalized supplementation with vitamin D3, in order to obtain optimized clinical benefits in the prevention of osteoporosis, sarcopenia, autoimmune diseases, and possibly different types of cancer. The importance of endogenous synthesis of vitamin D3 created an evolutionary pressure for reduced skin pigmentation, when, during the past 50,000 years, modern humans migrated from Africa towards Asia and Europe. This review will discuss different aspects of how vitamin D interacts with the human genome, focusing on nutritional epigenomics in context of immune responses. This should lead to a better understanding of the clinical benefits of vitamin D.

---

{{< toc >}}

#### Introduction

Nutrigenomics is a research field that developed some 15 years ago after the completion of the human genome <sup>[1,2]</sup>. Human diet is a complex mixture of biologically active molecules, some of which (i) have a direct effect on gene expression, (ii) modulate, after being metabolized, the activity of a transcription factor, or (iii) stimulate a signal transduction cascade that ends with the induction of a transcription factor <sup>[3]</sup>. Nutrigenomics aims to describe, characterize, and integrate the interactions between dietary compounds and genome-wide gene expression. Genomic loci related to metabolic and/or regulatory pathways may provide a molecular explanation of how a dietary compound mediates its effects and whether it influences the risk of diet-related diseases.

The discipline of epigenomics studies the effect of environmental events on the expression of the genome <sup>[4]</sup>. Diet is the main environmental exposure to the human body and represents the key daily lifestyle decision that every individual makes. Thus, a central goal of epigenomics is to get insight how dietary molecules affect the epigenome and by this gene expression <sup>[5]</sup>. Thus, nutrigenomics and epigenomics have a joint focus. Information provided by both fields may allow developing strategies to permit appropriate lifestyle decisions will lead to healthy, disease-free aging <sup>[6]</sup>.

Next-generation sequencing technologies, such as RNA sequencing, chromatin immunoprecipitation followed by sequencing (ChIP-seq), and formaldehyde-assisted identification of regulatory elements followed by sequencing (FAIRE-seq), allow the unbiased assessment of genome-wide mRNA expression, transcription factor binding, histone modifications, and chromatin accessibility <sup>[7]</sup>. Large research consortia, such as ENCODE (www.encodeproject.org) and Roadmap Epigenomics (www.roadmapepigenomics.org), collected huge amounts of data on the basal state of more than hundred human cell lines <sup>[8]</sup> as well as on about the same number of primary human tissues and cell types <sup>[9]</sup>, respectively.

Recent longitudinal molecular assessments of individuals in relation to their lifestyle decisions, such as the Personalized Omics Profiling Project <sup>[10,11]</sup> and the Pioneer 100 Wellness Project <sup>[12]</sup>, had a clear nutrigenomics component. When individuals are classified according to the features of their lifestyle, metabolic pathways, and (epi)genetic variation, tailored diets, referred to as personalized nutrition <sup>[13]</sup>, can be suggested for early therapeutic intervention. Interestingly, the use of dietary compounds for a specific therapy dissolves the distinction between food and drugs.

This review will discuss nutrigenomic aspects of vitamin D, which are based on epigenome- and transcriptome-wide actions of the micronutrient that were observed in vitro in human cell lines as well as in vivo in the context of human vitamin D intervention studies.

#### Vitamin D3

In the past, the primary medical focus on diet was to avoid nutrient deficiency diseases, such as scurvy and rickets, where patients lack the vitamins C and D, respectively. This is illustrated by the use of sunlight and ultraviolet (UV) radiation exposure as an efficient therapy of rickets, which is a childhood disease of bone malformation, as well as of tuberculosis, which is an infectious disease caused by the intracellular bacterium Mycobacterium tuberculosis. In parallel, some 100 years ago, the rickets curing ingredient of cod-liver oil was identified and termed "vitamin D", because it was the vitamin to be named fourth in line <sup>[14]</sup>. These observations were combined, when many years later it was found that humans can synthesize vitamin D3 in their UV-B (290-315 nm) exposed skin <sup>[15]</sup>. The energy of the UV radiation is used in a non-enzymatic reaction to open the B-ring of the cholesterol precursor 7-dehydrocholesterol, creating the thermodynamically unstable molecule pre-vitamin D3, which then further isomerizes into vitamin D3 (Figure 1A).

The ability to endogenously synthesize vitamin D3 implies that the term "vitamin" may not be correctly used. However, since nowadays humans tend to stay preferentially indoors and cover their skin by textile outdoors <sup>[16]</sup>, insufficient UV-B exposure results in low endogenous vitamin D3 production, so that for a huge proportion of the human population, the molecule is an essential micronutrient. Since the average human diet does not contain much vitamin D, worldwide, an estimated number of more than a billion individuals are vitamin D deficient <sup>[17]</sup>. Fatty fish, such as tuna, sardines, salmon, and mackerel, are the best dietary sources of vitamin D followed by liver, cheese, egg yolks, and mushrooms. Moreover, in some countries, foods, such as milk and other dairy products, orange juice, cereals, and margarines, are fortified with vitamin D. In addition, direct vitamin D supplementation via pills or oily drops is recommended in many countries <sup>[18]</sup>.

#### Vitamin D and Human Genetic Adaptation

Some 100,000 years ago, anatomically modern humans (Homo sapiens sapiens) lived only in Africa and their skin was dark, because it better protects against sunburns and skin cancer. Dark skin also prevents the photodegradation of circulating folate, which is an important methyl-group donor being important, e.g., for DNA methylation during embryogenesis. The latter aspect may havr been the main reason why, approximately one; million years ago, human skin became highly pigmented, when the uncestorf of modern humans lose most of their body hair, in order to better sweat during endurance physical activity, such as hunting <sup>[19]</sup>. Intereotingly, when modern humane arrived in Europe some 40,000 years ago, the reverse process happened, i.e., their skin turned pale again (Figure 1B).

The main evolutionary driver of this decreased skin pigmentation was ihe need Sor sufficient endogenouo vitamin Dr production ire less sunny Europe. The lightesa skin is found ini the Noodic countries, representing a geographic region where humans live most distant from the equator. Thus, vitamin D influences the human genome not only via VDR-mediated gene regulation, but did act as an evolutionary driver of genome adaptation in terms of skin lightening. Having dark skin when living permanently at northern latitudes would have caused vitamin D3 deficiency, resulting in reduced bone strength and a defective immune system. For example, nowadays, the tuberculosis rate of individuals with dark skin living distant from the equator is significantly higher than that of people with light skin <sup>[20]</sup>.

More than 30 genes are known to influence the pigmentation of melanocytes, but single nucleotide polymorphisms (SNPs) affecting the genes KITLG (encoding for a ligand of the receptor tyrosine kinase KIT), TYRP1 (a tyrosine converting enzyme), SLC24A5, and SLC45A2 (both are ion transporters) had the main impact on skin lightening during the past 10-30,000 years <sup>[21,22]</sup>. Interestingly, Neanderthals and modern Europeans and Asians evolved light skin independently from each other. Taken together, skin lightening due to the need for vitamin D3 synthesis in geographic regions with lower levels of UV-B radiation is a key example in nutrigenomics demonstrating how the need for a gene-regulating micronutrient has directed human evolution.

#### Vitamin D and Its Receptor in Metabolism and Immunity

In the liver, the biologically inert molecule vitamin D3 is converted to 25-hydroxyvitamin D3 (25(OH)D3, Figure 2A), which is the most stable and abundant vitamin D metabolite in human serum, and is traditionally used as a biomarker for the vitamin D status of individuals <sup>[23]</sup>. Further hydroxylation at carbon 1 creates 1a,25-dihydroxyvitamin D3 (1,25(OH)2D3, Figure 2A), which acts as an endocrine hormone by serving as a high-affinity ligand (kD 0.1 nM) to the transcription factor vitamin D receptor (VDR, Figure 2B) <sup>[24]</sup>. The main source of the endocrine production of 1,25(OH)2D3 are proximal tubule cells of the kidneys, but in a paracrine or autocrine fashion, monocytes, macrophages and dendritic cells of the innate immune system, osteoblasts within bones, and keratinocytes of the skin are also able to produce the hormone <sup>[25]</sup>.

The well-known physiological role of vitamin D is the support of bone mineralization via controlling calcium homeostasis (Figure 2C) <sup>[26]</sup>. This function fits well with the metabolism-focused profile of some members of the nuclear receptor superfamily, to which VDR also belongs <sup>[27]</sup>. Comparable members of the superfamily are retinoic acid receptor (RAR) a/p/y activated by the vitamin A derivative retinoic acid, peroxisome proliferator-activated receptor (PPAR) a/b/y binding fatty acids and liver X receptor (LXR) a/p recognizing oxysterols <sup>[28]</sup>. VDR, RAR, PPAR and LXR act as sensors of the respective micro- and macronutrients, in order to adapt gene expression profiles in metabolic organs as well as in cells of the immune system (Figure 2C). Since most immune cells have a rapid turnover, they are able to show a maximal adaptive response to environmental changes. Macrophages and dendritic cells, as well as their precursors, monocytes, coordinate metabolic, inflammatory, and general stress-response pathways via changes of their transcriptome profile and respective subtype specification <sup>[29]</sup>. Thus, lipid sensing and signaling via key nuclear receptors has an important role in the differentiation and epigenetic programming of immune cells, indicating that metabolism and immunity are more closely linked than initially assumed.

Via VDR, vitamin D modulates the activity of both the innate and adaptive immune system <sup>[30]</sup>. For example, in monocytes, vitamin D stimulates the efficient recognition of bacterial pathogens via toll-like receptors (TLRs) <sup>[31]</sup> and in macrophages, it inhibits the proliferation of M. tuberculosis <sup>[32]</sup>. Notably, tuberculosis is the most disastrous infectious disease that has ever occurred in humans, with an estimated death toll of more than one billion over the past 60,000 years <sup>[33]</sup>. TLR-triggered expression of vitamin D target genes encoding for the antimicrobial peptides cathelicidin and defensin beta 4A enables killing intracellular M. tuberculosis <sup>[34]</sup>. For this reason, sunlight or artificial UV-B exposure are efficient treatments of tuberculosis. In contrast, vitamin D deficiency is associated with an increased risk of developing clinical tuberculosis. Moreover, vitamin D has been shown to also prevent other type of microbial infections, such as those of the respiratory or urinary tract <sup>[35,36]</sup>.VDR is a key transcription factor in the differentiation process o, myelotd progenitors tnto monocytes and granulocytes i37]. Moreover, VDR and its ligands antagonize pro-inflammatory transcription faciors, such as NO-AT, AP-1, an, NF-kB, in T cells, which results in the decreased expression of cytok ines, such as IL2 and IL12 <sup>[38]</sup>. In addition, vita min ID» inhibits differentidtion, maturation, and the immunostimulatory capacity ot dendritic cells via the repression of ge nets encoding; for the dtfferent variants of the major hi stocompatibility complex and its co- stimulatory molecules CD40, CD80, and CD86 <sup>[39]</sup>. The resulting immune tolerance-inducing phenotype of dendritic cells leads to the ieduction of regulatory T cells down-regulating the activity other cells erf the immune system. This is the central mechanism of how vitamin D dampens chronic inflammation and autoimmunity in diseases such as inflammatory bowel disease <sup>[40]</sup> and multiple sclerosis <sup>[41]</sup>.

Interestingly, the imprinting of dendritic cells with tolerogenic properties involves the reprogramming of their glucose metabolism via up-regulation of the vitamin D target gene 6-phosphofructo-2-kinase/fructose-2,6-biphosphatase 4 (PFKFB4), which encodes for a key glycolytic enzyme <sup>[42]</sup>. Another glycolytic enzyme, fructose-bisphosphatase 1 (FBP1), is encoded by one of the most responsive primary vitamin D target genes of the monocytic cell line THP-1 <sup>[43]</sup>. This suggests that, also in vitamin D signaling, immunomodulatory and metabolic functions are more closely linked than initially assumed and probably evolved in parallel. Accordingly, the concept of immunometabolism suggests that metabolism is the key process determining the phenotype and function within immune cell subsets <sup>[44]</sup>.

 **Nutritional Epigenomics** 

The three-dimensional complex of genomic DNA and nucleosome-forming histone proteins is called chromatin <sup>[45]</sup>. It is categorized into less densely packed euchromatin, which easily accessible to transcription factors and other nuclear proteins, and compact heterochromatin, which is a functionally repressed state <sup>[46]</sup> (Figure 3A). Post-translational histone modifications and DNA methylations as well as changes of the 3-dimensional structure represent functionally relevant chromatin stages <sup>[47]</sup>. Euchromatin is located towards the center of the nucleus and in this open form of chromatin, histone proteins are mostly acetylated and genomic DNA is unmethylated. In contrast, heterochromatin is found closer to the nuclear membrane and in this closed chromatin form, both nucleosomes and genomic DNA are methylated <sup>[48]</sup>.

Epigenomics studies chromatin alterations that do not involve changes to the genome <sup>[49]</sup>. Epigenome changes, also referred to as epigenomic programing, are very prominent during embryogenesis, where totipotent stem cells beget various pluripotent cell lines of the embryo, which in turn act as precursors of terminally differentiated cells <sup>[50]</sup>. This differentiation process restricts the access to an increasing number of genomic regions and genes that they are controlling, so that terminally differentiated cells are able to focus on their specialized functions. Thus, chromatin accessibility plays an important role in regulating gene expression.

There is dynamic competition between nucleosomes and transcription factors for critical binding regions within genomic DNA, such as enhancers and promoters. Chromatin dynamics are influenced by a large set of chromatin modifying and remodeling enzymes, which interpret ("read"), add ("write"), or remove ("erase") post-translational histone modifications or DNA methylation <sup>[51]</sup>. Interestingly, the activity of many of these chromatin modifiers critically depends on intracellular levels of key intermediary metabolites, such as NAD+, acetyl-CoA, and a-ketoglutarate <sup>[52]</sup>. In this way, environmental inputs, such as the availability of energy substrates, have direct effects on the epigenome and, by this, on gene expression. This implies that chromatin modifiers act as sensors of metabolic information, such as cells being in a fasting or feeding state.

The field of nutritional epigenomics describes numerous connections between diet-derived metabolites and the epigenome. For example, a number of secondary metabolites from fruits, vegetables, spices, teas, and medicinal herbs, such as resveratrol, genistein, curcumin, and polyphenols, affect the activity of chromatin modifiers and transcription factors <sup>[53]</sup>. Vitamin D and other micro- and macronutrients affect via their nuclear receptor sensors of chromatin accessibility, i.e., they belong to field of nutritional epigenomics <sup>[54]</sup>. Importantly, in contrast to epigenomic programing for cell fate decisions during cellular differentiation, which is largely irreversible, diet-induced epigenomic changes are dynamic, i.e., they are often transient and reversible.

#### Epigenomics of Vitamin D

The established model of -vitamin D signaling <sup>[55]</sup> suggests that VDR, like RAR, PPAR, LXR, and other nuclear receptors <sup>[56]</sup>, form!? a heterodimer with the retinoid X receptor and preferentially binds to a DNA sequence that is formed of a direct regeat of two hexameric motifs spaced by three nucleotides, so-called DR3-type response elements <sup>[57]</sup>. The VDR cistrome <sup>[58]</sup>, as obtained in a number of human cell lines, confirmed on a genome-wide level that DR3-type sequences are the most prominent VDR binding motifs. ChIP-seq data from B lymphocytes <sup>[59]</sup>, monocytes <sup>[43,60]</sup>, colorectal cancer cells <sup>[61]</sup>, hepatic stellate cells <sup>[62]</sup>, and macrophage-like cells <sup>[63]</sup> indicate that in the absence of ligand, VDR only binds to some 200-2000 sites per cell type. After stimulation with 1,25(OH)2D3, the number of VDR sites increases in average by a factor of 2.5 <sup>[63]</sup>, which is the first direct epigenome-wide effect of vitamin D <sup>[54]</sup> (Figure 3B, I).

VDR is a "settler" type of transcription factor, i.e., it binds to a genomic region only when it finds its preferred sequence motifs within accessible chromatin. In contrast, "pioneer factors" <sup>[64]</sup> use DNA recognition sequences, which are short enough that they can be accessed even in the presence of nucleosomes. For example, VDR and the pioneer factors purine-rich box 1 (PU.1) and CCAAT/enhancer binding protein alpha (CEBPA) work closely together in hematopoietic differentiation into monocytes and granulocytes <sup>[37]</sup>. The pioneer factors PU.1 and CEBPA as well as GA binding protein transcription factor subunit alpha (GABPA) help VDR to find enhancer regions close to vitamin D target genes <span>[65-67]</span>. In THP-1 cells, 1,25(OH)2D3 modulates the binding of some 5-10% of PU.1, CEBPA, and GABPA binding sites, i.e., subsets of the cistromes of the respective pioneer factor are sensitive to the levels of the micronutrient. This represents the second level of epigenomic effects of vitamin D (Figure 3B, II).

At some 1300 genomic sites, the binding profile of the transcription factor CCCTC binding factor (CTCF) is also affected by 1,25(OH)2D3 <sup>[68]</sup>. CTCF is the key protein in organizing the human genome into loops, referred to as topologically associated domains (TADs) <sup>[69]</sup>. TADs are insulated from each other concerning gene regulatory effects, i.e., a transcription factor binding to an enhancer will modulate the expression of only those genes that have their transcription start sites (TSSs) within the same loop. In THP-1 cells, the vitamin D sensitivity of CTCF makes some 500 TADs dependent on VDR agonists, i.e., the modulation of the 3-dimensional organization of chromatin is the third level of the epigenome-wide effects of vitamin D (Figure 3B, III).

The high affinity binding of 1,25(OH)2D3 to the ligand-binding pocket within VDR's ligand-binding domain causes a conformational change in the receptor protein <sup>[70]</sup>. This results in the loss of VDR's contact with corepressor proteins, such as nuclear corepressor 1 (NCOR1) <sup>[71]</sup>, which, in the absence of ligand, link the receptor to chromatin modifiers of the histone deacetylase family. Instead, in the presence of ligand, VDR interacts with co-activator proteins, such as those of the nuclear co-activator (NCOA) family <sup>[72]</sup>, which either act themselves as histone acetyltransferases or form a complex with members of the respective chromatin modifier family. In addition, VDR communicates in a ligand-dependent fashion with chromatin modifiers of the lysine demethylase family, such as lysine demethylase 6B (KDM6B), or with chromatin remodelers like bromodomain containing 7 (BRD7). These proteins either form complexes with VDR <sup>[73,74]</sup> or their genes are primary targets of vitamin D <sup>[75]</sup>. The ligand-dependent interference of VDR with chromatin modifiers explains how vitamin D can affect histone markers for active TSSs, H3K4me3, and active chromatin, H3K27ac, as observed by ChIP-seq for these markers in 1,25(OH)2D3-stimulated THP-1 cells <sup>[66,76]</sup>. A change in histone modification has consequences on chromatin accessibility, as determined by FAIRE-seq <sup>[77]</sup>, and represents the fourth level of epigenomic effects of vitamin D (Figure 3B, IV).

ChIP-seq and FAIRE-seq results obtained in 1,25(OH)2D3-triggered THP-1 cells are, to date, the most comprehensive set of data on epigenome-wide effects of vitamin D and were the most influential in formulating the chromatin model of vitamin D signaling <sup>[78]</sup>. This model is presently challenged in other cellular systems, such as in peripheral blood mononuclear cells (PBMCs), which were obtained from vitamin D3 bolus supplemented individuals <sup>[79]</sup>.

#### Personalized Response to Vitamin D

The 1000 Genomes Project (www.internationalgenome.org) indicated that individuals differ from each other by in average by 4-5 million SNPs, some 0.5 million short insertions or deletions (indels), and up to a thousand larger copy number variations <sup>[80]</sup>. Although the majority of these variations are functionally neutral, there are many thousands of sites throughout the genome that increase the risk for common diseases or mediate the responsiveness to drugs. A well-known pharmacogenomics example of the latter is that of the anticoagulant drug warfarin, the individual's responsiveness dependent on SNPs of the genes encoding for the enzymes vitamin K epoxide reductase complex subunit 1 (VKORC1) and cytochrome P450 family 2C9 (CYP2C9) <sup>[81]</sup>. Accordingly, there are high, mid, and low responders to warfarin and based on their genotype, individuals are prescribed different doses of the drug.

Recently, an analogous concept came up for vitamin D <sup>[82]</sup>. Based on the vitamin D3 supplementation studies VitDmet (NCT01479933, ClinicalTrials.gov) <span>[83-86]</span> and VitDbol (NCT02063334) <sup>[87,88]</sup>, individuals show a personalized response to vitamin D3 and can be segregated into high, mid, and low responders. In contrast to in vitro cell culture experiments, supplementation studies have the advantage that they assess the activity of vitamin D under in vivo conditions. Thus, both types of intervention studies represent in vivo human nutrigenomic experiments. VitDmet was a long-term intervention study using daily vitamin D3 supplementation (0-80 pg) of pre-diabetic elderly individuals over 5 months of a Finnish winter, while VitDbol assessed the effects of a single vitamin D3 bolus (2000 ug) in young healthy adults already after one day. In both studies, PBMCs were isolated before and after supplementation and RNA and chromatin was prepared without any further in vitro culture of the cells. The index was initially determined on the basis of the response of 24 primary vitamin D target genes and 12 vitamin D sensitive biochemical parameters, such parathyroid hormone serum levels <sup>[86]</sup> as well as on the vitamin D-dependent change of the accessibility of a few chromatin regions <sup>[88]</sup>, but most accurately, it was assessed by on the transcriptome-wide response of 702 vitamin D target genes (Figure 4, top right) <sup>[89]</sup>. In relation to fold changes in 25(OH)D3 serum levels, the individuals were ranked and classified into the three groups of vitamin D responsiveness. Both studies, VitDmet and VitDbol, indicated that some 25% of the participants are low responders, i.e., they display a low vitamin D response index <sup>[16]</sup>. Low vitamin D responders are most vulnerable against vitamin D deficiency In order to keep vitamin D endocrinology at its optimal activity, low vitamin D responders need to take far higher daily vitamin D3 doses (maybe 50-100 pg) than high responders (10-20 pg).

The vitamin D response index was found to be independent of the 25(OH)D3 serum levels, i.e., of the vitamin D status of the study participants. In analogy to the warfarin responsiveness, it is assumed that the vitamin D response index is an intrinsic property of each individual and does not change during lifetime <sup>[82]</sup>. However, it is not yet known which genomic and epigenomic variations determine the index. The nutrigenomic assessment of in vivo vitamin D responsiveness, i.e., the determination of the vitamin D response index, implies that there should be a personalized vitamin D3 supplementation advice, rather than a general, population-based recommendation. Accordingly, the physiological responses to vitamin D on the level of bone and muscle health as well as on the prevention of autoimmune diseases and possibly different types of cancer (breast, prostate, and colon) are expected to be maximal (Figure 4, bottom).

Vitamin D3 (endogenous conversion to 25(OH)D3 and 1,25(OH)2D3)

 **Prevention of osteoporosis, sarcopenia, autoimmune responses and cancer** 

Figure 4. Clinical benefits of optimized vitamin D action. The vitamin D response index segregates individuals into high, mid, and low vitamin D responders and allows a more accurate monitoring of vitamin D effects in clinical setups, such as a prevention of osteoporosis, sarcopenia, autoimmune diseases, and possibly even cancer. The index is based on individual's genetic and epigenetic status, but does not depend on their vitamin D status. It is determined primarily via changes of the transcriptome (i.e., mRNA transcription of vitamin D target genes) of vitamin D responding tissues, such as PBMCs. FC, fold change.

#### Clinical Impact of Vitamin D Nutrigenomics

As outlined above, besides controlling proper bone mineralization, vitamin D's main physiological role seems to be to the modulation of responses of the innate and adaptive immune responses. During the past years, a number of clinical studies demonstrated the immune system related benefit of vitamin D. For example, vitamin D deficiency in pregnancy may cause early asthma and wheezing, since vitamin D has effects on the developing lung and immune system during the fetal and early postnatal periods <sup>[90]</sup>. Therefore, vitamin D3 supplementation during pregnancy and infancy is important, in order to prevent aeroallergen sensitization, such as the proportion of children sensitized to mites, and the occurrence of respiratory illnesses, such as asthma <sup>[91]</sup>. Other randomized control trials indicated that vitamin D3 supplementation during pregnancy decreased the rate of respiratory tract infections or wheeze within the first 5 years of infant's life <sup>[92]</sup>. Thus, a personalized vitamin D3 supplementation after vitamin D response index determination may reduce the predisposition toward developing allergic hypersensitivity reactions leading to asthma and/or respiratory tract infections more efficiently than relying exclusively on the vitamin D status. Moreover, vitamin D response index-based stratification of participants of future vitamin D intervention studies should lead to more convincing results concerning the benefits of vitamin D than reported recently for the risk of cancer and cardiovascular disease <sup>[93]</sup>.

#### Conclusions

Molecular insights on nutrient-sensing systems allow a more integrative view of the reaction of the human body to dietary molecules. Vitamin D and its metabolites belong to a small set of dietary compounds that have direct effects on gene regulation. The analysis of vitamin D signaling via next generation sequencing technologies in in vitro cell culture models as well as in primary cells, such as PBMCs, has generated large amounts of data on the vitamin D-triggered epigenome and transcriptome in the respective cellular systems. Since the micronutrient connects cellular metabolism with immunity <sup>[94,95]</sup> (Figure 2C), nutrigenomics of vitamin D has a pleiotropic physiological and clinical impact. Personalized nutrition, such as tailored vitamin D supplementation, will contribute to the maintenance of wellbeing and the prevention of age- and lifestyle-related diseases, in particular those related to chronic inflammation.

#### References

1. Lander, E.S.; Linton, L.M.; Birren, B.; Nusbaum, C.; Zody, M.C.; Baldwin, J.; Devon, K.; Dewar, K.; Doyle, M.; FitzHugh, W.; et al. Initial sequencing and analysis of the human genome. Nature 2001, 409, 860-921. <span>[PubMed]</span>

1. International Human Genome Sequencing Consortium. Finishing the euchromatic sequence of the human genome. Nature 2004, 431, 931-945.

1. Müller, M.; Kersten, S. Nutrigenomics: Goals and strategies. Nat Rev Genet 2003, 4, 315-322. <span>[CrossRef]</span>

1. Meissner, A. What can epigenomics do for you? Genome Biol 2012,13, 420. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Ulven, S.M.; Molnar, F. Nutrigenomics; Springer Textbook; Springer: Berlin, Germany, 2016.

1.  **Carlberg, C** .; Molnar, F. Human Epigenomics; Springer Textbook: Berlin, Germany, 2018.

1. Meyer, C.A.; Liu, X.S. Identifying and mitigating bias in next-generation sequencing methods for chromatin biology. Nat. Rev. Genet. 2014,15, 709-721. <span>[CrossRef]</span> <span>[PubMed]</span>

1. ENCODE-Project-Consortium; Bernstein, B.E.; Birney, E.; Dunham, I.; Green, E.D.; Gunter, C.; Snyder, M. An integrated encyclopedia of DNA elements in the human genome. Nature 2012, 489, 57-74.

1. Roadmap Epigenomics, C.; Kundaje, A.; Meuleman, W.; Ernst, J.; Bilenky, M.; Yen, A.; Heravi-Moussavi, A.; Kheradpour, P.; Zhang, Z.; Wang, J.; et al. Integrative analysis of 111 reference human epigenomes. Nature 2015, 518, 317-330.

1. Chen, R.; Mias, G.I.; Li-Pook-Than, J.; Jiang, L.; Lam, H.Y.; Chen, R.; Miriami, E.; Karczewski, K.J.; Hariharan, M.; Dewey, F.E.; et al. Personal omics profiling reveals dynamic molecular and medical phenotypes. Cell 2012,148,1293-1307. <span>[CrossRef]</span>

1. Jiang, C.; Wang, X.; Li, X.; Inlora, J.; Wang, T.; Liu, Q.; Snyder, M. Dynamic human environmental exposome revealed by longitudinal personal monitoring. Cell 2018,175, 277-291 e31. <span>[CrossRef]</span>

1. Price, N.D.; Magis, A.T.; Earls, J.C.; Glusman, G.; Levy, R.; Lausted, C.; McDonald, D.T.; Kusebauch, U.; Moss, C.L.; Zhou, Y.; et al. A wellness study of 108 individuals using personal, dense, dynamic data clouds. Nat. Biotechnol. 2017, 35, 747-756. <span>[CrossRef]</span>

1. Kaput, J. Nutrigenomics research for personalized nutrition and medicine. Curr. Opin. Biotechnol. 2008,19, 110-120. <span>[CrossRef]</span>

1. McMollum, E.V.; Simmonds, N.; Becker, J.E.; Shipley, P.G. Studies on experimental rickets: An experimental demonstration of the existence of a vitamin which promotes calcium deposition. J. Biol. Chem. 1922, 52, 293-298.

1. Holick, M.F. The cutaneous photosynthesis of previtamin D3: A unique photoendocrine system. J. Investig. Dermatol. 1981, 77, 51-58. <span>[CrossRef]</span>

1.  **Carlberg, C** . Molecular approaches for optimizing vitamin D supplementation. Vitam. Horm. 2016, 100, 255-271.

1. Holick, M.F. Vitamin D deficiency. N. Engl. J. Med. 2007, 357, 266-281. <span>[CrossRef]</span>

1. Bendik, I.; Friedel, A.; Roos, F.F.; Weber, P.; Eggersdorfer, M. Vitamin D: A critical and essential micronutrient for human health. Front. Physiol. 2014, 5, 248. <span>[CrossRef]</span>

1. Jablonski, N.G.; Chaplin, G. The roles of vitamin D and cutaneous vitamin D production in human evolution and health. Int. J. Paleopathol. 2018,23, 54-59. <span>[CrossRef]</span>

1. Pilarski, A.; Penn, N.; Ratnakumar, S.; Barker, R.D.; Milburn, H.J. Variation in vitamin D deficiency among tuberculosis patients by ethnic group and geographical region of birth: Evidence from a diverse south London population. Eur. Respir. J. 2016, 48,1507-1510. <span>[CrossRef]</span>

1. Quillen, E.E.; Norton, H.L.; Parra, E.J.; Lona-Durazo, F.; Ang, K.C.; Illiescu, F.M.; Pearson, L.N.; Shriver, M.D.; Lasisi, T.; Gokcumen, O.; et al. Shades of complexity: New perspectives on the evolution and genetic architecture of human skin. Am. J. Phys. Anthropol. 2019,168 (Suppl. 67), 4-26. <span>[CrossRef]</span>

1. Deng, L.; Xu, S. Adaptation of human skin color in various populations. Hereditas 2018,155,1. <span>[CrossRef]</span>

1. Hollis, B.W. Circulating 25-hydroxyvitamin D levels indicative of vitamin D sufficiency: Implications for establishing a new effective dietary intake recommendation for vitamin D. J. Nutr. 2005, 135, 317-322. <span>[CrossRef]</span>

1. Mangelsdorf, D.J.; Thummel, C.; Beato, M.; Herrlich, P.; Schütz, G.; Umesono, K.; Blumberg, B.; Kastner, P.; Mark, M.; Chambon, P.; et al. The nuclear receptor superfamily: The second decade. Cell 1995, 83, 835-839. <span>[CrossRef]</span>

1. Hewison, M.; Burke, F.; Evans, K.N.; Lammas, D.A.; Sansom, D.M.; Liu, P.; Modlin, R.L.; Adams, J.S. Extra-renal 25-hydroxyvitamin Ö3-1a-hydroxylase in human health and disease. J. Steroid Biochem. Mol. Biol. 2007, 103, 316-321. <span>[CrossRef]</span>

1. Bouillon, R.; Suda, T. Vitamin D: Calcium and bone homeostasis during evolution. BoneKEy Rep. 2014, 3, 480. <span>[CrossRef]</span>

1. Sonoda, J.; Pei, L.; Evans, R.M. Nuclear receptors: Decoding metabolic disease. FEBS Lett. 2008, 582, 2-9. <span>[CrossRef]</span>

1. Evans, R.M. The nuclear receptor superfamily: A rosetta stone for physiology. Mol. Endocrinol. 2005,19, 1429-1438. <span>[CrossRef]</span>

1. Hoeksema, M.A.; de Winther, M.P. Epigenetic regulation of monocyte and macrophage function. Antioxid. Redox Signal. 2016, 25, 758-774. <span>[CrossRef]</span>

1. Prietl, B.; Treiber, G.; Pieber, T.R.; Amrein, K. Vitamin D and immune function. Nutrients 2013,5, 2502-2521. <span>[CrossRef]</span>

1. Liu, P.T.; Stenger, S.; Li, H.; Wenzel, L.; Tan, B.H.; Krutzik, S.R.; Ochoa, M.T.; Schauber, J.; Wu, K.; Meinken, C.; et al. Toll-like receptor triggering of a vitamin D-mediated human antimicrobial response. Science 2006, 311, 1770-1773. <span>[CrossRef]</span>

1. Rook, G.A.; Steele, J.; Fraher, L.; Barker, S.; Karmali, R.; O'Riordan, J.; Stanford, J. Vitamin D3, gamma interferon, and control of proliferation of Mycobacterium tuberculosis by human monocytes. Immunology 1986, 57,159-163.

1. Paulson, T. Epidemiology: A mortal foe. Nature 2013, 502, S2-S3. <span>[CrossRef]</span>

1. Gombart, A.F.; Borregaard, N.; Koeffler, H.P. Human cathelicidin antimicrobial peptide (CAMP) gene is a direct target of the vitamin D receptor and is strongly up-regulated in myeloid cells by 1,25-dihydroxyvitamin D3. FASEB J. 2005,19,1067-1077. <span>[CrossRef]</span>

1. Jorde, R.; Sollid, S.T.; Svartberg, J.; Joakimsen, R.M.; Grimnes, G.; Hutchinson, M.Y. Prevention of urinary tract infections with vitamin D supplementation 20,000 IU per week for five years. Results from an RCT including 511 subjects. Infect. Dis. (Lond.) 2016, 48, 823-828.

1. Kearns, M.D.; Alvarez, J.A.; Seidel, N.; Tangpricha, V. Impact of vitamin D on infectious disease. Am. J. Med. Sci. 2015, 349, 245-262. <span>[CrossRef]</span>

1. Novershtern, N.; Subramanian, A.; Lawton, L.N.; Mak, R.H.; Haining, W.N.; McConkey, M.E.; Habib, N.; Yosef, N.; Chang, C.Y.; Shay, T.; et al. Densely interconnected transcriptional circuits control cell states in human hematopoiesis. Cell 2011,144, 296-309. <span>[CrossRef]</span>

1. Zeitelhofer, M.; Adzemovic, M.Z.; Gomez-Cabrero, D.; Bergman, P.; Hochmeister, S.; N'Diaye, M.; Paulson, A.; Ruhrmann, S.; Almgren, M.; Tegner, J.N.; et al. Functional genomics analysis of vitamin D effects on CD4+ T cells in vivo in experimental autoimmune encephalomyelitis. Proc. Natl. Acad. Sci. USA 2017, 114, E1678-E1687. <span>[CrossRef]</span>

1. Barragan, M.; Good, M.; Kolls, J.K. Regulation of dendritic cell function by vitamin D. Nutrients 2015, 7, 8127-8151. <span>[CrossRef]</span>

1. Limketkai, B.N.; Mullin, G.E.; Limsui, D.; Parian, A.M. Role of vitamin D in inflammatory bowel disease. Nutr. Clin. Pract. 2017, 32, 337-345. <span>[CrossRef]</span>

1. Munger, K.L.; Levin, L.I.; Hollis, B.W.; Howard, N.S.; Ascherio, A. Serum 25-hydroxyvitamin D levels and risk of multiple sclerosis. JAMA 2006, 296, 2832-2838. <span>[CrossRef]</span>

1. Vanherwegen, A.S.; Eelen, G.; Ferreira, G.B.; Ghesquiere, B.; Cook, D.P.; Nikolic, T.; Roep, B.; Carmeliet, P.; Telang, S.; Mathieu, C.; et al. Vitamin D controls the capacity of human dendritic cells to induce functional regulatory T cells by regulation of glucose metabolism. J. Steroid Biochem. Mol. Biol. 2019, 187, 134-145. <span>[CrossRef]</span>

1. Heikkinen, S.; Vaisanen, S.; Pehkonen, P.; Seuter, S.; Benes, V.;  **Carlberg, C** . Nuclear hormone 1 a,25-dihydroxyvitamin D3 elicits a genome-wide shift in the locations of VDR chromatin occupancy. Nucleic Acids Res. 2011, 39, 9181-9193. <span>[CrossRef]</span>

1. Vanherwegen, A.S.; Gysemans, C.; Mathieu, C. Vitamin D endocrinology on the cross-road between immunity and metabolism. Mol. Cell Endocrinol. 2017, 453, 52-67. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Molnar, F. The Impact of Chromatin. Mechanisms of Gene Regulation, 2nd ed.; Springer Textbook: Berlin, Germany, 2016; pp. 17-34.

1. Beisel, C.; Paro, R. Silencing chromatin: Comparing modes and mechanisms. Nat. Rev. Genet. 2011, 12, 123-135. <span>[CrossRef]</span>

1. Rivera, C.M. Ren, B. Mapping Human Epigenomes. Cell 2013,155, 39-55.

1. Hathaway, N.A.; Bell, O.; Hodges, C.; Miller, E.L.; Neel, D.S.; Crabtree, G.R. Dynamics and memory of heterochromatin in living cells. Cell 2012,149,1447-1460. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Molnar, F. The epigenome. Mechanisms of Gene Regulation, 2nd ed.; Springer Textbook: Berlin, Germany, 2016; pp. 159-172.

1. Perino, M.; Veenstra, G.J. Chromatin control of developmental dynamics and plasticity. Dev. Cell 2016, 38, 610-620. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Molnar, F. Chromatin Modifiers. Mechanisms of Gene Regulation, 2nd ed.; Springer Textbook: Berlin, Germany, 2016; pp. 129-145.

1. Gut, P.; Verdin, E. The nexus of chromatin regulation and intermediary metabolism. Nature 2013, 502, 489-498. <span>[CrossRef]</span>

1. Vanden Berghe, W. Epigenetic impact of dietary polyphenols in cancer chemoprevention: Lifelong remodeling of our epigenomes. Pharmacol. Res. 2012, 65, 565-576. <span>[CrossRef]</span>

1.  **Carlberg, C** . Molecular endocrinology of vitamin D on the epigenome level. Mol. Cell. Endocrinol. 2017, 453, 14-21. <span>[CrossRef]</span>

1. Pike, J.W.; Meyer, M.B.; Watanuki, M.; Kim, S.; Zella, L.A.; Fretz, J.A.; Yamazaki, M.; Shevde, N.K. Perspectives on mechanisms of gene regulation by 1,25-dihydroxyvitamin D3 and its receptor. J. Steroid Biochem. Mol. Biol. 2007,103, 389-395. <span>[CrossRef]</span>

1. Mangelsdorf, D.J.; Evans, R.M. The RXR heterodimers and orphan receptors. Cell 1995, 83, 841-850. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Bendik, I.; Wyss, A.; Meier, E.; Sturzenbecker, L.J.; Grippo, J.F.; Hunziker, W. Two nuclear signalling pathways for vitamin D. Nature 1993, 361, 657-660. <span>[CrossRef]</span> <span>[PubMed]</span>

1.  **Carlberg, C** . Genome-wide (over)view on the actions of vitamin D. Front. Physiol. 2014, 5,167. <span>[CrossRef]</span> <span>[PubMed]</span>

1. Ramagopalan, S.V.; Heger, A.; Berlanga, A.J.; Maugeri, N.J.; Lincoln, M.R.; Burrell, A.; Handunnetthi, L.; Handel, A.E.; Disanto, G.; Orton, S.M.; et al. A ChIP-seq defined genome-wide map of vitamin D receptor binding: Associations with disease and evolution. Genome Res. 2010, 20,1352-1360. <span>[CrossRef]</span> <span>[PubMed]</span>

1. Neme, A.; Seuter, S.;  **Carlberg, C** . Selective regulation of biological processes by vitamin D based on the spatio-temporal cistrome of its receptor. Biochim. Biophys. Acta 2017,1860, 952-961. <span>[CrossRef]</span> <span>[PubMed]</span>

1. Meyer, M.B.; Goetsch, P.D.; Pike, J.W. VDR/RXR and TCF4/beta-catenin cistromes in colonic cells of colorectal tumor origin: Impact on c-FOS and c-MYC gene expression. Mol. Endocrinol. 2012, 26, 37-51. <span>[CrossRef]</span> <span>[PubMed]</span>

1. Ding, N.; Yu, R.T.; Subramaniam, N.; Sherman, M.H.; Wilson, C.; Rao, R.; Leblanc, M.; Coulter, S.; He, M.; Scott, C.; et al. A vitamin D receptor/SMAD genomic circuit gates hepatic fibrotic response. Cell 2013,153, 601-613. <span>[CrossRef]</span> <span>[PubMed]</span>

1. Tuoresmaki, P.; Vaisanen, S.; Neme, A.; Heikkinen, S.;  **Carlberg, C** . Patterns of genome-wide VDR locations. PLoS ONE 2014, 9, e96105. <span>[CrossRef]</span>

1. Zaret, K.S.; Carroll, J.S. Pioneer transcription factors: Establishing competence for gene expression. Gen. Dev. 2011, 25,2227-2241. <span>[CrossRef]</span>

1. Seuter, S.; Neme, A.;  **Carlberg, C** . Epigenomic PU.1-VDR crosstalk modulates vitamin D signaling. Biochim. Biophys. Acta 2017,1860, 405-415.

1. Nurminen, V.; Neme, A.; Seuter, S.;  **Carlberg, C** . Modulation of vitamin D signaling by the pioneer factor CEBPA. Biochim. Biophys. Acta 2019,1862, 96-106. <span>[CrossRef]</span>

1. Seuter, S.; Neme, A.;  **Carlberg, C** . ETS transcription factor family member GABPA contributes to vitamin D receptor target gene regulation. J. Steroid Biochem. Mol. Biol. 2018,177, 46-52. <span>[CrossRef]</span>

1. Neme, A.; Seuter, S.;  **Carlberg, C** . Vitamin D-dependent chromatin association of CTCF in human monocytes. Biochim. Biophys. Acta 2016,1859,1380-1388. <span>[CrossRef]</span>

1. Dixon, J.R.; Selvaraj, S.; Yue, F.; Kim, A.; Li, Y.; Shen, Y.; Hu, M.; Liu, J.S.; Ren, B. Topological domains in mammalian genomes identified by analysis of chromatin interactions. Nature 2012, 485, 376-380. <span>[CrossRef]</span>

1. Molnar, F.; Perakyla, M.;  **Carlberg, C** . Vitamin D receptor agonists specifically modulate the volume of the ligand-binding pocket. J. Biol. Chem. 2006, 281,10516-10526. <span>[CrossRef]</span>

1. Polly, P.; Herdick, M.; Moehren, U.; Baniahmad, A.; Heinzel, T.;  **Carlberg, C** . VDR-Alien: A novel, DNA-selective vitamin D3 receptor-corepressor partnership. Faseb J 2000,14,1455-1463. <span>[CrossRef]</span>

1. Herdick, M.;  **Carlberg, C** . Agonist-triggered modulation of the activated and silent state of the vitamin D3 receptor by interaction with co-repressors and co-activators. J. Mol. Biol. 2000, 304, 793-801. <span>[CrossRef]</span>

1. Molnar, F. Structural considerations of vitamin D signaling. Front. Physiol. 2014, 5,191. <span>[CrossRef]</span>

1. Wei, Z.; Yoshihara, E.; He, N.; Hah, N.; Fan, W.; Pinto, A.F.M.; Huddy, T.; Wang, Y.; Ross, B.; Estepa, G.; et al. Vitamin D switches BAF complexes to protect beta cells. Cell 2018,173,1135-1149 e15. <span>[CrossRef]</span>

1. Pereira, F.; Barbachano, A.; Silva, J.; Bonilla, F.; Campbell, M.J.; Munoz, A.; Larriba, M.J. KDM6B/JMJD3 histone demethylase is induced by vitamin D and modulates its effects in colon cancer cells. Hum. Mol. Genet. 2011, 20, 4655-4665. <span>[CrossRef]</span>

1. Nurminen, V.; Neme, A.; Seuter, S.;  **Carlberg, C** . The impact of the vitamin D-modulated epigenome on VDR target gene regulation. Biochim. Biophys. Acta 2018,1861, 697-705. <span>[CrossRef]</span>

1. Seuter, S.; Neme, A.;  **Carlberg, C** . Epigenome-wide effects of vitamin D and their impact on the transcriptome of human monocytes involve CTCF. Nucleic Acids Res. 2016, 44, 4090-4104. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Neme, A. Machine learning approaches infer vitamin D signaling: Critical impact of vitamin D receptor binding within topologically associated domains. J. Steroid Biochem. Mol. Biol. 2019,185,103-109. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Seuter, S.; Nurmi, T.; Tuomainen, T.P.; Virtanen, J.K.; Neme, A. In vivo response of the human epigenome to vitamin D: A proof-of-principle study. J. Steroid Biochem. Mol. Biol. 2018, 180, 142-148. <span>[CrossRef]</span>

1. Genomes Project, C.; Abecasis, G.R.; Auton, A.; Brooks, L.D.; DePristo, M.A.; Durbin, R.M.; Handsaker, R.E.; Kang, H.M.; Marth, G.T.; McVean, G.A. An integrated map of genetic variation from 1,092 human genomes. Nature 2012, 491, 56-65.

1. Rieder, M.J.; Reiner, A.P.; Gage, B.F.; Nickerson, D.A.; Eby, C.S.; McLeod, H.L.; Blough, D.K.; Thummel, K.E.; Veenstra, D.L.; Rettie, A.E. Effect of VKORC1 haplotypes on transcriptional regulation and warfarin dose. N. Engl. J. Med. 2005, 352, 2285-2293. <span>[CrossRef]</span>

1.  **Carlberg, C** .; Haq, A. The concept of the personal vitamin D response index. J. Steroid Biochem. Mol. Biol. 2018,175,12-17. <span>[CrossRef]</span>

> See VitaminDWiki [Huge variation in response to vitamin D supplementation – personal vitamin D response index – Dec 2016](/posts/huge-variation-in-response-to-vitamin-d-supplementation-personal-vitamin-d-response-index)

1.  **Carlberg, C** .; Seuter, S.; de Mello, V.D.; Schwab, U.; Voutilainen, S.; Pulkki, K.; Nurmi, T.; Virtanen, J.; Tuomainen, T.P.; Uusitupa, M. Primary vitamin D target genes allow a categorization of possible benefits of vitamin D3 supplementation. PLoS ONE 2013, 8, e71042. <span>[CrossRef]</span>

1. Wilfinger, J.; Seuter, S.; Tuomainen, T.-P.; Virtanen, J.K.; Voutilainen, S.; Nurmi, T.; de Mello, V.D.F.; Uusitupa, M.;  **Carlberg, C** . Primary vitamin D receptor target genes as biomarkers for the vitamin D3 status in the hematopoietic system. J. Nutr. Biochem. 2014, 25, 875-884. <span>[CrossRef]</span>

1. Ryynanen, J.; Neme, A.; Tuomainen, T.P.; Virtanen, J.K.; Voutilainen, S.; Nurmi, T.; de Mello, V.D.; Uusitupa, M.;  **Carlberg, C** . Changes in vitamin D target gene expression in adipose tissue monitor the vitamin D response of human individuals. Mol. Nutr. Food Res. 2014, 58, 2036-2045. <span>[CrossRef]</span>

1. Saksa, N.; Neme, A.; Ryynanen, J.; Uusitupa, M.; de Mello, V.D.; Voutilainen, S.; Nurmi, T.; Virtanen, J.K.; Tuomainen, T.P.;  **Carlberg, C** . Dissecting high from low responders in a vitamin D3 intervention study. J. Steroid Biochem. Mol. Biol. 2015,148, 275-282. <span>[CrossRef]</span>

1. Vukic, M.; Neme, A.; Seuter, S.; Saksa, N.; de Mello, V.D.; Nurmi, T.; Uusitupa, M.; Tuomainen, T.P.; Virtanen, J.K.;  **Carlberg, C** . Relevance of vitamin D receptor target genes for monitoring the vitamin D responsiveness of primary human cells. PLoS ONE 2015,10, e0124339. <span>[CrossRef]</span>

1. Seuter, S.; Virtanen, J.K.; Nurmi, T.; Pihlajamaki, J.; Mursu, J.; Voutilainen, S.; Tuomainen, T.P.; Neme, A.;  **Carlberg, C** . Molecular evaluation of vitamin D responsiveness of healthy young adults. J. Steroid Biochem. Mol. Biol. 2017,174,314-321. <span>[CrossRef]</span>

1. Neme, A.; Seuter, S.; Malinen, M.; Nurmi, T.; Tuomainen, T.P.; Virtanen, J.K.;  **Carlberg, C** . In vivo transcriptome changes of human white blood cells in response to vitamin D. J. Steroid Biochem. Mol. Biol. 2019,188, 71-76. <span>[CrossRef]</span>

1. Litonjua, A.A.; Carey, V.J.; Laranjo, N.; Harshfield, B.J.; McElrath, T.F.; O'Connor, G.T.; Sandel, M.; Iverson, R.E., Jr.; Lee-Paritz, A.; Strunk, R.C.; et al. Effect of prenatal supplementation with vitamin D on asthma or recurrent wheezing in offspring by age 3 years: The VDAART randomized clinical trial. JAMA 2016, 315, 362-370. <span>[CrossRef]</span>

1. Grant, C.C.; Crane, J.; Mitchell, E.A.; Sinclair, J.; Stewart, A.; Milne, T.; Knight, J.; Gilchrist, C.; Camargo, C.A., Jr. Vitamin D supplementation during pregnancy and infancy reduces aeroallergen sensitization: A randomized controlled trial. Allergy 2016, 71,1325-1334. <span>[CrossRef]</span>

1. Christensen, N.; Sondergaard, J.; Fisker, N.; Christesen, H.T. Infant respiratory tract infections or wheeze and maternal vitamin D in pregnancy: A systematic review. Pediatr. Infect. Dis. J. 2017, 36, 384-391. <span>[CrossRef]</span>

1. Manson, J.E.; Cook, N.R.; Lee, I.M.; Christen, W.; Bassuk, S.S.; Mora, S.; Gibson, H.; Gordon, D.; Copeland, T.; D'Agostino, D.; et al. Vitamin D supplements and prevention of cancer and cardiovascular disease. N. Engl. J. Med. 2019, 380, 33-44. <span>[CrossRef]</span>

1. Verstuyf, A.; Carmeliet, G.; Bouillon, R.; Mathieu, C. Vitamin D: A pleiotropic hormone. Kidney Int. 2010, 78, 140-145. <span>[CrossRef]</span>

1.  **Carlberg, C** . The physiology of vitamin D-far more than calcium and bone. Front. Physiol. 2014, 5, 335. <span>[CrossRef]</span>